<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    		<meta name="keywords" content="Bayesian inference in phylogeny,Phylo,Phylo,Bayesian inference,Cladistics,Computational phylogenetics,DNA barcoding,Least squares inference in phylogeny,List of evolutionary biology topics,Long branch attraction,Markov Chain" />
		<link rel="shortcut icon" href="/favicon.ico" />
		<link rel="search" type="application/opensearchdescription+xml" href="/w/opensearch_desc.php" title="Wikipedia (English)" />
		<link rel="copyright" href="../../../COPYING.html" />
    <title>Bayesian inference in phylogeny - Wikipedia, the free encyclopedia</title>
    <style type="text/css">/*<![CDATA[*/ @import "../../../skins/htmldump/main.css"; /*]]>*/</style>
    <link rel="stylesheet" type="text/css" media="print" href="../../../skins/common/commonPrint.css" />
    <!--[if lt IE 5.5000]><style type="text/css">@import "../../../skins/monobook/IE50Fixes.css";</style><![endif]-->
    <!--[if IE 5.5000]><style type="text/css">@import "../../../skins/monobook/IE55Fixes.css";</style><![endif]-->
    <!--[if IE 6]><style type="text/css">@import "../../../skins/monobook/IE60Fixes.css";</style><![endif]-->
    <!--[if IE]><script type="text/javascript" src="../../../skins/common/IEFixes.js"></script>
    <meta http-equiv="imagetoolbar" content="no" /><![endif]-->
    <script type="text/javascript" src="../../../skins/common/wikibits.js"></script>
    <script type="text/javascript" src="../../../skins/htmldump/md5.js"></script>
    <script type="text/javascript" src="../../../skins/htmldump/utf8.js"></script>
    <script type="text/javascript" src="../../../skins/htmldump/lookup.js"></script>
    <script type="text/javascript" src="../../../raw/gen.js"></script>        <style type="text/css">/*<![CDATA[*/
@import "../../../raw/MediaWiki%7ECommon.css";
@import "../../../raw/MediaWiki%7EMonobook.css";
@import "../../../raw/gen.css";
/*]]>*/</style>          </head>
  <body
    class="ns-0">
    <div id="globalWrapper">
      <div id="column-content">
	<div id="content">
	  <a name="top" id="contentTop"></a>
	        <h1 class="firstHeading">Bayesian inference in phylogeny</h1>
	  <div id="bodyContent">
	    <h3 id="siteSub">From Wikipedia, the free encyclopedia</h3>
	    <div id="contentSub"></div>
	    	    	    <!-- start content -->
	    <p><b>Bayesian inference in phylogeny</b> generates a posterior distribution for a parameter, composed of a <a href="../../../p/h/y/Phylogenetic_tree.html" title="Phylogenetic tree">phylogenetic tree</a> and a model of evolution, based on the prior for that parameter and the likelihood of the data, generated by a multiple alignment. The Bayesian approach has become more popular due to advances in computational machinery, especially, <a href="../../../m/a/r/Markov_Chain_52f1.html" title="Markov Chain">Markov Chain</a> <a href="../../../m/o/n/Monte_Carlo_9de6.html" title="Monte Carlo">Monte Carlo</a> algorithms. <a href="../../../b/a/y/Bayesian_inference.html" title="Bayesian inference">Bayesian inference</a> has a number of applications in <a href="../../../m/o/l/Molecular_phylogenetics.html" title="Molecular phylogenetics">molecular phylogenetics</a>, for example, estimation of <a href="../../../s/p/e/Species.html" title="Species">species</a> phylogeny and species divergence times.</p>
<table id="toc" class="toc" summary="Contents">
<tr>
<td>
<div id="toctitle">
<h2>Contents</h2>
</div>
<ul>
<li class="toclevel-1"><a href="#Basic_Bayesian_Theory"><span class="tocnumber">1</span> <span class="toctext">Basic Bayesian Theory</span></a></li>
<li class="toclevel-1"><a href="#The_LOCAL_algorithm_of_Larget_and_Simon"><span class="tocnumber">2</span> <span class="toctext">The LOCAL algorithm of Larget and Simon</span></a>
<ul>
<li class="toclevel-2"><a href="#Assessing_Convergence"><span class="tocnumber">2.1</span> <span class="toctext">Assessing Convergence</span></a></li>
</ul>
</li>
<li class="toclevel-1"><a href="#Metropolis-coupled_MCMC_.28Geyer.29"><span class="tocnumber">3</span> <span class="toctext">Metropolis-coupled MCMC (Geyer)</span></a></li>
<li class="toclevel-1"><a href="#References"><span class="tocnumber">4</span> <span class="toctext">References</span></a></li>
</ul>
</td>
</tr>
</table>
<script type="text/javascript">
//<![CDATA[
 if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } 
//]]>
</script>
<p><a name="Basic_Bayesian_Theory" id="Basic_Bayesian_Theory"></a></p>
<h2><span class="editsection">[<a href="../../../b/a/y/Bayesian_inference_in_phylogeny.html" title="Edit section: Basic Bayesian Theory">edit</a>]</span> <span class="mw-headline">Basic Bayesian Theory</span></h2>
<p>Recall that for Bayesian inference:</p>
<p><img class='tex' src="../../../math/0/5/5/05530bc9773b7919121327626c990f9e.png" alt="p(\theta | D) = \frac{p(D|\theta)p(\theta)}{p(D)}\" /></p>
<p>The denominator <img class='tex' src="../../../math/e/0/b/e0b94490f0679d548fe621c619121f05.png" alt="p(D)\" />is the <i>marginal probability of the data</i>, averaged over all possible parameter values weighted by their prior distribution. Formally,</p>
<p><img class='tex' src="../../../math/d/4/9/d49c0ecad7ead56221e97101b60bc8d6.png" alt="p(D) = \int_{\Theta}p(D|\theta)p(\theta)d\theta\" /></p>
<p>where <img class='tex' src="../../../math/0/3/3/033c14b6041145a9f77f67116f18d83d.png" alt="\Theta\" /> is the parameter space for <img class='tex' src="../../../math/4/8/7/48713aea6b053c91de9021b424b784a3.png" alt="\theta\" />.</p>
<p>In the original Metropolis algorithm, given a current <img class='tex' src="../../../math/4/8/7/48713aea6b053c91de9021b424b784a3.png" alt="\theta\" />-value <img class='tex' src="../../../math/b/1/9/b19dfd822e2517400335d53c1556df72.png" alt="x\" />, and a new <img class='tex' src="../../../math/4/8/7/48713aea6b053c91de9021b424b784a3.png" alt="\theta\" />-value <img class='tex' src="../../../math/e/d/f/edffd1a73cb003cdba3d07600e9aa8f2.png" alt="y\" />, the new value is accepted with probability:</p>
<dl>
<dd>
<dl>
<dd><img class='tex' src="../../../math/9/d/8/9d8b9034b2a7fe019f0e35327df82356.png" alt="h(y)/h(x) = \frac{p(D|y)p(y)}{p(D|x)p(x)}\" /></dd>
</dl>
</dd>
</dl>
<p><a name="The_LOCAL_algorithm_of_Larget_and_Simon" id="The_LOCAL_algorithm_of_Larget_and_Simon"></a></p>
<h2><span class="editsection">[<a href="../../../b/a/y/Bayesian_inference_in_phylogeny.html" title="Edit section: The LOCAL algorithm of Larget and Simon">edit</a>]</span> <span class="mw-headline">The LOCAL algorithm of Larget and Simon</span></h2>
<p>The LOCAL algorithm begins by selecting an internal branch of the tree at random. The nodes at the ends of this branch are each connected to two other branches. One of each pair is chosen at random. Imagine taking these three selected edges and stringing them like a clothesline from left to right, where the direction (left/right) is also selected at random. The two endpoints of the first branch selected will have a sub-tree hanging like a piece of clothing strung to the line. The algorithm proceeds by multiplying the three selected branches by a common random amount, akin to stretching or shrinking the clothesline. Finally the leftmost of the two hanging sub-trees is disconnected and reattached to the clothesline at a location selected uniformly at random. <i>This is the candidate tree</i>.</p>
<p>Suppose we began by selecting the internal branch with length $$<img class='tex' src="../../../math/f/8/f/f8f6ebc6f797f55bad3b270cb861529e.png" alt="t_8\" /> (in Figure (a) (to be added)) that separates taxa <img class='tex' src="../../../math/f/b/6/fb6b20dfdc28097525ae99b3bd2c8a23.png" alt="A\" /> and <img class='tex' src="../../../math/6/2/2/622305a4d4aa133a26c5b53ff4a3c729.png" alt="B\" /> from the rest. Suppose also that we have (randomly) selected branches with lengths <img class='tex' src="../../../math/0/2/2/02266b7efd0e3d8c2d30a10f4a396214.png" alt="t_1\" /> and <img class='tex' src="../../../math/3/e/5/3e514ba66d021039bdc2d0b5e81206ca.png" alt="t_9\" /> from each side, and that we oriented these branches as shown in Figure(b). Let <img class='tex' src="../../../math/6/e/e/6ee66af16365546a276010412c0dedfc.png" alt="m = t_1+t_8+t_9\" />, be the current length of the clothesline. We select the new length to be <img class='tex' src="../../../math/4/1/c/41cb23833c1f8e48f0629fba23f65f63.png" alt="m^{\star} = m\exp(\lambda(U_1-0.5))\" />, where <img class='tex' src="../../../math/7/3/e/73e38a1155ef2b8802cc7ac857d52142.png" alt="U_1\" /> is a uniform random variable on <img class='tex' src="../../../math/e/3/4/e34cf57d80511f1621221bc748629623.png" alt="(0,1)\" />. Then for the LOCAL algorithm, the acceptance probability can be computed to be:</p>
<dl>
<dd>
<dl>
<dd><img class='tex' src="../../../math/1/9/0/19092794d86109343e0ce826d2e91e99.png" alt="\frac{h(y)}{h(x)} \times \frac{{m^{\star}}^3}{m^3}\" /></dd>
</dl>
</dd>
</dl>
<p><a name="Assessing_Convergence" id="Assessing_Convergence"></a></p>
<h3><span class="editsection">[<a href="../../../b/a/y/Bayesian_inference_in_phylogeny.html" title="Edit section: Assessing Convergence">edit</a>]</span> <span class="mw-headline">Assessing Convergence</span></h3>
<p>Suppose we want to estimate a branch length of a 2-taxon tree under JC, in which <span class="texhtml"><i>n</i><sub>1</sub></span> sites are unvaried and <span class="texhtml"><i>n</i><sub>2</sub></span> are variable. Assume exponential prior distribution with rate <img class='tex' src="../../../math/b/2/d/b2dc9f1f8ca4c6e87560b293915d8c22.png" alt="\lambda\" />. The density is <img class='tex' src="../../../math/8/4/f/84f03a47275af365c020a284cc56f542.png" alt="p(t) = \lambda e^{-\lambda t}\" />. The probabilities of the possible site patterns are:</p>
<dl>
<dd>
<dl>
<dd><img class='tex' src="../../../math/2/3/7/2371ca24592d55d827c04d40c2b3aa82.png" alt="1/4\left(1/4+3/4e^{-4/3t}\right)\" /></dd>
</dl>
</dd>
</dl>
<p>for unvaried sites, and</p>
<dl>
<dd>
<dl>
<dd><img class='tex' src="../../../math/1/d/c/1dcd4158261c698bed4b1a64c7fd78f0.png" alt="1/4\left(1/4-1/4e^{-4/3t}\right)\" /></dd>
</dl>
</dd>
</dl>
<p>Thus the unnormalized posterior distribution is:</p>
<dl>
<dd><img class='tex' src="../../../math/f/a/7/fa76e7d91dcea63061b418669c1c0e66.png" alt="h(t) =  \left(1/4\right)^{n_1+n_2}\left(1/4+3/4{e^{-4/3t}}^{n_1}\right)\" /></dd>
</dl>
<p>or, alternately,</p>
<dl>
<dd><img class='tex' src="../../../math/f/9/c/f9c165e7ba62b3350478926bfcc35b03.png" alt="h(t) = \left(1/4-1/4{e^{-4/3t}}^{n_2}\right)(\lambda e^{-\lambda t}\" /></dd>
</dl>
<p>Update branch length by choosing new value uniformly at random from a window of half-width <img class='tex' src="../../../math/8/6/3/86355e1622461adac0031526b4c5dd5a.png" alt="w\" /> centered at the current value:</p>
<dl>
<dd><img class='tex' src="../../../math/a/7/1/a717c1e9ba8f4d30e11d0ed96e535b24.png" alt="t^{\star} = |t+U|\" /></dd>
</dl>
<p>where <img class='tex' src="../../../math/2/0/1/2018ee47b429e6afa4a5aa175dde10c5.png" alt="U\" />is uniformly distributed between <img class='tex' src="../../../math/2/6/c/26c329ef8376fca1edddae81491cce2d.png" alt="-w\" /> and <img class='tex' src="../../../math/8/6/3/86355e1622461adac0031526b4c5dd5a.png" alt="w\" />. The acceptance probability is:</p>
<dl>
<dd><img class='tex' src="../../../math/f/8/9/f898452dc5c33cb15b08b7ec11d4d189.png" alt="h(t^{\star})/h(t)\" /></dd>
</dl>
<p>Example: <img class='tex' src="../../../math/e/6/9/e6928d6c7e9c04987978d14ade4cf052.png" alt="n_1 = 70\" />, <img class='tex' src="../../../math/f/3/f/f3f4afc2d030a5d4218c37cc2d2c83a0.png" alt="n_2 = 30\" />. We will compare results for two values of <img class='tex' src="../../../math/8/6/3/86355e1622461adac0031526b4c5dd5a.png" alt="w\" />, <img class='tex' src="../../../math/1/0/0/1009bddec21c6167ac277ad84ea173f0.png" alt="w = 0.1\" /> and <img class='tex' src="../../../math/4/f/c/4fcddc33b4da7978473e7c62bc0100b7.png" alt="w = 0.5\" />. In each case, we will begin with an initial length of <img class='tex' src="../../../math/e/4/c/e4c715bfb625dbbbd5a3f1480d09d10b.png" alt="5\" /> and update the length <img class='tex' src="../../../math/8/0/2/80226ba5aa068be17a1604945e0f0ce2.png" alt="2000\" /> times. (See Figure 3.2 (to be added) for results.)</p>
<p><a name="Metropolis-coupled_MCMC_.28Geyer.29" id="Metropolis-coupled_MCMC_.28Geyer.29"></a></p>
<h2><span class="editsection">[<a href="../../../b/a/y/Bayesian_inference_in_phylogeny.html" title="Edit section: Metropolis-coupled MCMC (Geyer)">edit</a>]</span> <span class="mw-headline">Metropolis-coupled MCMC (Geyer)</span></h2>
<p>If the target distribution has multiple peaks, separated by low valleys, the Markov chain may have difficulty in moving from one peak to another. As a result, the chain may get stuck on one peak and the resulting samples will not approximate the posterior density correctly. This is a serious practical concern for phylogeny reconstruction, as multiple local peaks are known to exist in the tree space during heuristic tree search under maximum parsimony (MP), maximum likelihood (ML), and minimum evolution (ME) criteria, and the same can be expected for stochastic tree search using MCMC. Many strategies have been proposed to improve mixing of Markov chains in presence of multiple local peaks in the posterior density. One of the most successful algorithms is the Metropolis-coupled MCMC (or <img class='tex' src="../../../math/c/0/a/c0a67147bfd79422086fff5c20688773.png" alt="\mathrm{MC}^3\" />).</p>
<p>In this algorithm, $m$ chains are run in parallel, with different stationary distributions <img class='tex' src="../../../math/f/1/c/f1c8f366735bd77335e6c15124a1f334.png" alt="\pi_j(.)\" />, $$<img class='tex' src="../../../math/0/2/4/024665c88a7453bead17132a08512c3e.png" alt="j = 1, 2, \ldots, m\" />, where the first one, <img class='tex' src="../../../math/4/8/4/4848df0bd79fc79c8b349fe92d4b7622.png" alt="\pi_1 = \pi\" /> is the target density, while <img class='tex' src="../../../math/5/c/4/5c4e8a51f9722254f8edee0c3c26069d.png" alt="\pi_j\" />, <img class='tex' src="../../../math/f/7/a/f7aea3f1b91aa635a4a1a779746137d4.png" alt="j = 2, 3, \ldots, m\" /> are chosen to improve mixing. For example, one can choose incremental heating of the form:</p>
<dl>
<dd>
<dl>
<dd><img class='tex' src="../../../math/0/a/9/0a92c55cd0f1382ba3204894312ec843.png" alt="\pi_j(\theta) = \pi(\theta)^{1/[1+\lambda(j-1)]}, \ \ \lambda &gt; 0," /></dd>
</dl>
</dd>
</dl>
<p>so that the first chain is the cold chain with the correct target density, while chains <img class='tex' src="../../../math/9/c/2/9c2b29e59a0d61f42c1b2d48f390d2cd.png" alt="2, 3, \ldots, m" /> are heated chains. Note that raising the density <span class="texhtml">π(.)</span> to the power <img class='tex' src="../../../math/4/e/b/4eb3994d2134ab3e1a72ec36cb90e16a.png" alt="1/T\" /> with <img class='tex' src="../../../math/b/3/2/b3271ab515cb4bd333b5eee184a689d5.png" alt="T&gt;1\" /> has the effect of flattening out the distribution, similar to heating a metal. In such a distribution, it is easier to traverse between peaks (separated by valleys) than in the original distribution. After each iteration, a swap of states between two randomly chosen chains is proposed through a Metropolis-type step. Let <img class='tex' src="../../../math/9/d/0/9d0426395675262bd5bd62cac6340c12.png" alt="\theta^{(j)}\" /> be the current state in chain <img class='tex' src="../../../math/3/0/9/3093e47b4a2ca4c3ad65231467ec5696.png" alt="j\" />, <img class='tex' src="../../../math/0/2/4/024665c88a7453bead17132a08512c3e.png" alt="j = 1, 2, \ldots, m\" />. A swap between the states of chains <img class='tex' src="../../../math/6/0/7/607708482f5a2afc26174c21834baad3.png" alt="i\" /> and <img class='tex' src="../../../math/3/0/9/3093e47b4a2ca4c3ad65231467ec5696.png" alt="j\" /> is accepted with probability:</p>
<dl>
<dd><img class='tex' src="../../../math/3/3/6/336b6e8fc6b6c554acd7503debcce4cd.png" alt="\alpha = \frac{\pi_i(\theta^{(j)}\pi_j(\theta^{(i)})}{\pi_i(\theta^{(i)})\pi_j(\theta^{(j)})}\" /></dd>
</dl>
<p>At the end of the run, output from only the cold chain is used, while those from the hot chains are discarded. Heuristically, the hot chains will visit the local peaks rather easily, and swapping states between chains will let the cold chain occasionally jump valleys, leading to better mixing. However, if <img class='tex' src="../../../math/1/5/7/15771ed3e7f6654c57c5a8c1fc1299d2.png" alt="\pi_i(\theta)/\pi_j(\theta)\" /> is unstable, proposed swaps will seldom be accepted. This is the reason for using several chains which differ only incrementally. (See Figure3.3 (to be added)).</p>
<p>An obvious disadvantage of the algorithm is that <img class='tex' src="../../../math/d/4/8/d482d61667a2a8f3a40b25f2626b6d16.png" alt="m\" /> chains are run and only one chain is used for inference. For this reason, <img class='tex' src="../../../math/c/0/a/c0a67147bfd79422086fff5c20688773.png" alt="\mathrm{MC}^3\" /> is ideally suited for implementation on parallel machines, since each chain will in general require the same amount of computation per iteration.</p>
<p><br /></p>
<p><a name="References" id="References"></a></p>
<h2><span class="editsection">[<a href="../../../b/a/y/Bayesian_inference_in_phylogeny.html" title="Edit section: References">edit</a>]</span> <span class="mw-headline">References</span></h2>
<ul>
<li>Geyer, C.J. (1991) Markov chain Monte Carlo maximum likelihood. In <i>Computing Science and Statistics: Proceedings of the 23rd Symposium of the Interface</i> (ed. E.M. Keramidas), pp. 156-163. Interface Foundation, Fairfax Station, VA.</li>
<li>Yang, Z. and B. Rannala. (1997) Bayesian phylogenetic inference using DNA sequences: A Markov chain Monte Carlo method. <i>Molecular Biology and Evolution</i>, <b>14</b>, 717-724.</li>
<li>Larget, B. and D.L. Simon. (1999) Markov chain Monte Carlo algorithms for the Bayesian analysis of phylogenetic trees. <i>Molecular Biology and Evolution</i>, <b>16</b>, 750-759.</li>
<li>Huelsenbeck, J.P. and F. Ronquist. (2001) MrBayes: Bayesian inference in phylogenetic trees. <i>Bioinformatics</i>, <b>17</b>, 754-755.</li>
<li>Ronquist, F. and J.P. Huelsenbeck. (2003) MrBayes3: Bayesian phylogenetic inference under mixed models. <i>Bioinformatics</i>, <b>19</b>, 1572-1574.</li>
<li>Rannala, B. and Z. Yang. (2003) Bayes estimation of species divergence times and ancestral population sizes using DNA sequences from multiple loci. <i>Genetics</i>, <b>164</b>, 1645-1656.</li>
</ul>
<table style="margin:0.5em auto; text-align:center; clear:both" align="center" class="toccolours">
<tr>
<th style="background:#ccccff">
<div style="text-align: center; position: relative;">Topics in phylogenetics
<div class="noprint plainlinksneverexpand" style="background-color:transparent; padding:0; font-weight:normal; font-size:xx-small; color:#000000; white-space:nowrap; position: absolute; right: 0; top: 0; margin: 2px; font-weight: normal;"><a href="../../../p/h/y/Template%7EPhylo_9a91.html" title="Template:Phylo"><span title="View this template">v</span></a>&#160;<span style="font-size:80%;">•</span>&#160;<a href="../../../p/h/y/Template_talk%7EPhylo_9a8a.html" title="Template talk:Phylo"><span title="Discussion about this template">d</span></a>&#160;<span style="font-size:80%;">•</span>&#160;<a href="http://en.wikipedia.org../../../p/h/y/Template%7EPhylo_9a91.html" class="external text" title="http://en.wikipedia.org../../../p/h/y/Template%7EPhylo_9a91.html" rel="nofollow"><span style="color: #002bb8;" title="You can edit this template. Please use the preview button before saving.">e</span></a></div>
</div>
</th>
</tr>
<tr>
<td style="font-size:90%"><b>Relevant fields</b>: <a href="../../../p/h/y/Phylogenetics.html" title="Phylogenetics">phylogenetics</a> | <a href="../../../c/o/m/Computational_phylogenetics.html" title="Computational phylogenetics">computational phylogenetics</a> | <a href="../../../m/o/l/Molecular_phylogeny.html" title="Molecular phylogeny">molecular phylogeny</a> | <a href="../../../c/l/a/Cladistics.html" title="Cladistics">cladistics</a></td>
</tr>
<tr>
<td style="font-size:90%"><b>Basic concepts</b>: <a href="../../../s/y/n/Synapomorphy.html" title="Synapomorphy">synapomorphy</a> | <a href="../../../p/h/y/Phylogenetic_tree.html" title="Phylogenetic tree">phylogenetic tree</a> | <a href="../../../p/h/y/Phylogenetic_network.html" title="Phylogenetic network">phylogenetic network</a> | <a href="../../../l/o/n/Long_branch_attraction.html" title="Long branch attraction">long branch attraction</a></td>
</tr>
<tr>
<td style="font-size:90%"><b>Phylogeny inference methods</b>: <a href="../../../m/a/x/Maximum_parsimony.html" title="Maximum parsimony">maximum parsimony</a> | <a href="../../../m/a/x/Maximum_likelihood.html" title="Maximum likelihood">maximum likelihood</a> | <a href="../../../n/e/i/Neighbour_joining.html" title="Neighbour joining">neighbour joining</a> | <a href="../../../u/p/g/UPGMA_789f.html" title="UPGMA">UPGMA</a> | <strong class="selflink">Bayesian inference</strong> | <a href="../../../l/e/a/Least_squares_inference_in_phylogeny.html" title="Least squares inference in phylogeny">Least Squares</a></td>
</tr>
<tr>
<td style="font-size:90%"><b>Current topics</b>: <a href="../../../p/h/y/PhyloCode_a482.html" title="PhyloCode">PhyloCode</a> | <a href="../../../d/n/a/DNA_barcoding_dc89.html" title="DNA barcoding">DNA barcoding</a></td>
</tr>
<tr>
<td style="font-size:90%"><a href="../../../l/i/s/List_of_evolutionary_biology_topics.html" title="List of evolutionary biology topics">List of evolutionary biology topics</a></td>
</tr>
</table>

<!-- 
Pre-expand include size: 5027 bytes
Post-expand include size: 3831 bytes
Template argument size: 353 bytes
Maximum: 2048000 bytes
-->
<div class="printfooter">
Retrieved from "<a href="http://en.wikipedia.org../../../b/a/y/Bayesian_inference_in_phylogeny.html">http://en.wikipedia.org../../../b/a/y/Bayesian_inference_in_phylogeny.html</a>"</div>
	    <div id="catlinks"><p class='catlinks'><a href="../../../c/a/t/Special%7ECategories_101d.html" title="Special:Categories">Categories</a>: <span dir='ltr'><a href="../../../b/i/o/Category%7EBioinformatics_ee23.html" title="Category:Bioinformatics">Bioinformatics</a></span> | <span dir='ltr'><a href="../../../p/h/y/Category%7EPhylogenetics_52eb.html" title="Category:Phylogenetics">Phylogenetics</a></span> | <span dir='ltr'><a href="../../../c/o/m/Category%7EComputational_phylogenetics_b639.html" title="Category:Computational phylogenetics">Computational phylogenetics</a></span></p></div>	    <!-- end content -->
	    <div class="visualClear"></div>
	  </div>
	</div>
      </div>
      <div id="column-one">
	<div id="p-cactions" class="portlet">
	  <h5>Views</h5>
	  <ul>
	    <li id="ca-nstab-main"
	       class="selected"	       ><a href="../../../b/a/y/Bayesian_inference_in_phylogeny.html">Article</a></li><li id="ca-talk"
	       	       ><a href="../../../b/a/y/Talk%7EBayesian_inference_in_phylogeny_1ae8.html">Discussion</a></li><li id="ca-current"
	       	       ><a href="http://en.wikipedia.org/wiki/Bayesian_inference_in_phylogeny">Current revision</a></li>	  </ul>
	</div>
	<div class="portlet" id="p-logo">
	  <a style="background-image: url(../../../images/wiki-en.png);"
	    href="../../../index.html"
	    title="Main Page"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
		<div class='portlet' id='p-navigation'>
	  <h5>Navigation</h5>
	  <div class='pBody'>
	    <ul>
	    	      <li id="n-Main-page"><a href="../../../index.html">Main page</a></li>
	     	      <li id="n-Contents"><a href="../../../c/o/n/Wikipedia%7EContents_3181.html">Contents</a></li>
	     	      <li id="n-Featured-content"><a href="../../../f/e/a/Wikipedia%7EFeatured_content_24ba.html">Featured content</a></li>
	     	      <li id="n-currentevents"><a href="../../../c/u/r/Portal%7ECurrent_events_bb60.html">Current events</a></li>
	     	    </ul>
	  </div>
	</div>
		<div class='portlet' id='p-interaction'>
	  <h5>interaction</h5>
	  <div class='pBody'>
	    <ul>
	    	      <li id="n-About-Wikipedia"><a href="../../../a/b/o/Wikipedia%7EAbout_8d82.html">About Wikipedia</a></li>
	     	      <li id="n-portal"><a href="../../../c/o/m/Wikipedia%7ECommunity_Portal_6a3c.html">Community portal</a></li>
	     	      <li id="n-contact"><a href="../../../c/o/n/Wikipedia%7EContact_us_afd6.html">Contact us</a></li>
	     	      <li id="n-sitesupport"><a href="http://wikimediafoundation.org/wiki/Fundraising">Make a donation</a></li>
	     	      <li id="n-help"><a href="../../../c/o/n/Help%7EContents_22de.html">Help</a></li>
	     	    </ul>
	  </div>
	</div>
		<div id="p-search" class="portlet">
	  <h5><label for="searchInput">Search</label></h5>
	  <div id="searchBody" class="pBody">
	    <form action="javascript:goToStatic(3)" id="searchform"><div>
	      <input id="searchInput" name="search" type="text"
	        accesskey="f" value="" />
	      <input type='submit' name="go" class="searchButton" id="searchGoButton"
	        value="Go" />
	    </div></form>
	  </div>
	</div>
	      </div><!-- end of the left (by default at least) column -->
      <div class="visualClear"></div>
      <div id="footer">
    <div id="f-poweredbyico"><a href="http://www.mediawiki.org/"><img src="../../../skins/common/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" /></a></div>	<div id="f-copyrightico"><a href="http://wikimediafoundation.org/"><img src="../../../images/wikimedia-button.png" border="0" alt="Wikimedia Foundation"/></a></div>	<ul id="f-list">
	  	  	  <li id="f-credits">This page was last modified 19:15, 4 March 2007 by Wikipedia user <a href="../../../p/a/c/User%7EPacaro_4866.html" title="User:Pacaro">Pacaro</a>. Based on work by Wikipedia user(s) <a href="../../../s/k/a/User%7ESkapur_f77a.html" title="User:Skapur">Skapur</a>, <a href="../../../s/a/n/User%7ESanjay_Tiwari_1173.html" title="User:Sanjay Tiwari">Sanjay Tiwari</a> and <a href="../../../e/l/o/User%7EElonka_d4b2.html" title="User:Elonka">Elonka</a>.</li>	  <li id="f-copyright">All text is available under the terms of the <a class='internal' href="../../../t/e/x/Wikipedia%7EText_of_the_GNU_Free_Documentation_License_702a.html" title="Wikipedia:Text of the GNU Free Documentation License">GNU Free Documentation License</a>. (See <b><a class='internal' href="../../../c/o/p/Wikipedia%7ECopyrights_92c4.html" title="Wikipedia:Copyrights">Copyrights</a></b> for details.) <br /> Wikipedia&reg; is a registered trademark of the <a href="http://www.wikimediafoundation.org">Wikimedia Foundation, Inc</a>., a US-registered <a class='internal' href="../../../5/0/1/501%28c%29.html#501.28c.29.283.29" title="501(c)(3)">501(c)(3)</a> <a href="http://wikimediafoundation.org/wiki/Deductibility_of_donations">tax-deductible</a> <a class='internal' href="../../../n/o/n/Non-profit_organization.html" title="Non-profit organization">nonprofit</a> <a href="../../../c/h/a/Charitable_organization.html" title="Charitable organization">charity</a>.<br /></li>	  <li id="f-about"><a href="../../../a/b/o/Wikipedia%7EAbout_8d82.html" title="Wikipedia:About">About Wikipedia</a></li>	  <li id="f-disclaimer"><a href="../../../g/e/n/Wikipedia%7EGeneral_disclaimer_3e44.html" title="Wikipedia:General disclaimer">Disclaimers</a></li>	  	</ul>
      </div>
    </div>
  </body>
</html>
