<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    		<meta name="keywords" content="Bisulfite sequencing,5-methylcytosine,Adenine,Agarose,Allele,Amplicon,Amplicons,Antisense,Bisulfite,Cancers,Cloning" />
		<link rel="shortcut icon" href="/favicon.ico" />
		<link rel="search" type="application/opensearchdescription+xml" href="/w/opensearch_desc.php" title="Wikipedia (English)" />
		<link rel="copyright" href="../../../COPYING.html" />
    <title>Bisulfite sequencing - Wikipedia, the free encyclopedia</title>
    <style type="text/css">/*<![CDATA[*/ @import "../../../skins/htmldump/main.css"; /*]]>*/</style>
    <link rel="stylesheet" type="text/css" media="print" href="../../../skins/common/commonPrint.css" />
    <!--[if lt IE 5.5000]><style type="text/css">@import "../../../skins/monobook/IE50Fixes.css";</style><![endif]-->
    <!--[if IE 5.5000]><style type="text/css">@import "../../../skins/monobook/IE55Fixes.css";</style><![endif]-->
    <!--[if IE 6]><style type="text/css">@import "../../../skins/monobook/IE60Fixes.css";</style><![endif]-->
    <!--[if IE]><script type="text/javascript" src="../../../skins/common/IEFixes.js"></script>
    <meta http-equiv="imagetoolbar" content="no" /><![endif]-->
    <script type="text/javascript" src="../../../skins/common/wikibits.js"></script>
    <script type="text/javascript" src="../../../skins/htmldump/md5.js"></script>
    <script type="text/javascript" src="../../../skins/htmldump/utf8.js"></script>
    <script type="text/javascript" src="../../../skins/htmldump/lookup.js"></script>
    <script type="text/javascript" src="../../../raw/gen.js"></script>        <style type="text/css">/*<![CDATA[*/
@import "../../../raw/MediaWiki%7ECommon.css";
@import "../../../raw/MediaWiki%7EMonobook.css";
@import "../../../raw/gen.css";
/*]]>*/</style>          </head>
  <body
    class="ns-0">
    <div id="globalWrapper">
      <div id="column-content">
	<div id="content">
	  <a name="top" id="contentTop"></a>
	        <h1 class="firstHeading">Bisulfite sequencing</h1>
	  <div id="bodyContent">
	    <h3 id="siteSub">From Wikipedia, the free encyclopedia</h3>
	    <div id="contentSub"></div>
	    	    <div class="usermessage">You have <a href="../../../1/2/7/User_talk%7E127.0.0.1.html" title="User talk:127.0.0.1">new messages</a> (<a href="../../../1/2/7/User_talk%7E127.0.0.1.html" title="User talk:127.0.0.1">last change</a>).</div>	    <!-- start content -->
	    <div class="thumb tright">
<div class="thumbinner" style="width:302px;"><a href="../../../w/i/k/Image%7EWiki_Bisulfite_sequencing_Figure_1_small.png_78a3.html" class="internal" title="Figure 1: Outline of bisulfite conversion of sample sequence of genomic DNA.  Nucleotides in blue are unmethylated cytosines converted to uracils by bisulfite, while red nucleotides are 5-methylcytosines resistant to conversion."><img src="../../../upload/thumb/c/c9/Wiki_Bisulfite_sequencing_Figure_1_small.png/300px-Wiki_Bisulfite_sequencing_Figure_1_small.png" alt="Figure 1: Outline of bisulfite conversion of sample sequence of genomic DNA.  Nucleotides in blue are unmethylated cytosines converted to uracils by bisulfite, while red nucleotides are 5-methylcytosines resistant to conversion." width="300" height="209" longdesc="../../../w/i/k/Image%7EWiki_Bisulfite_sequencing_Figure_1_small.png_78a3.html" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="../../../w/i/k/Image%7EWiki_Bisulfite_sequencing_Figure_1_small.png_78a3.html" class="internal" title="Enlarge"><img src="../../../skins/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
Figure 1: Outline of bisulfite conversion of sample sequence of genomic DNA. Nucleotides in blue are unmethylated cytosines converted to uracils by bisulfite, while red nucleotides are 5-methylcytosines resistant to conversion.</div>
</div>
</div>
<p><b>Bisulfite sequencing</b> is the use of <a href="../../../b/i/s/Bisulfite.html" title="Bisulfite">bisulfite</a> treatment of <a href="../../../d/n/a/DNA_396c.html" title="DNA">DNA</a> to determine its pattern of methylation. <a href="../../../d/n/a/DNA_methylation_b466.html" title="DNA methylation">DNA methylation</a> was the first discovered <a href="../../../e/p/i/Epigenetic.html" title="Epigenetic">epigenetic</a> mark, and remains the most studied. In animals it predominantly involves the addition of a <a href="../../../m/e/t/Methyl_group.html" title="Methyl group">methyl group</a> to the carbon-5 position of <a href="../../../c/y/t/Cytosine.html" title="Cytosine">cytosine</a> residues of the dinucleotide <a href="../../../c/p/g/CpG_e9e8.html" title="CpG">CpG</a>, and is implicated in repression of genetic activity.</p>
<p>Bisulfite sequencing is based on the fact that treatment of DNA with <a href="../../../b/i/s/Bisulfite.html" title="Bisulfite">bisulfite</a> converts <a href="../../../c/y/t/Cytosine.html" title="Cytosine">cytosine</a> residues to <a href="../../../u/r/a/Uracil.html" title="Uracil">uracil</a>, but leaves <a href="../../../5/-/m/5-methylcytosine.html" title="5-methylcytosine">5-methylcytosine</a> residues unaffected. Bisulfite treatment thus introduces specific changes in the <a href="../../../d/n/a/DNA_sequence_dd40.html" title="DNA sequence">DNA sequence</a> that depend on the <a href="../../../m/e/t/Methylation.html" title="Methylation">methylation</a> status of individual cytosine residues, yielding very high-resolution information about the methylation status of a segment of DNA. Various analyses can be performed on the altered sequence to retrieve this information. The objective of this analysis is therefore reduced to differentiating between <a href="../../../s/i/n/Single_nucleotide_polymorphisms.html" title="Single nucleotide polymorphisms">single nucleotide polymorphisms</a> (cytosines and thymines) resulting from bisulfite conversion (Figure 1).</p>
<table id="toc" class="toc" summary="Contents">
<tr>
<td>
<div id="toctitle">
<h2>Contents</h2>
</div>
<ul>
<li class="toclevel-1"><a href="#Methods"><span class="tocnumber">1</span> <span class="toctext">Methods</span></a>
<ul>
<li class="toclevel-2"><a href="#1._Non-methylation-specific_PCR_based_methods"><span class="tocnumber">1.1</span> <span class="toctext">1. Non-methylation-specific PCR based methods</span></a>
<ul>
<li class="toclevel-3"><a href="#a.29_Direct_sequencing"><span class="tocnumber">1.1.1</span> <span class="toctext">a) Direct sequencing</span></a></li>
<li class="toclevel-3"><a href="#b.29_Pyrosequencing"><span class="tocnumber">1.1.2</span> <span class="toctext">b) Pyrosequencing</span></a></li>
<li class="toclevel-3"><a href="#c.29_Methylation-sensitive_single-strand_conformation_analysis.28MS-SSCA.29"><span class="tocnumber">1.1.3</span> <span class="toctext">c) Methylation-sensitive single-strand conformation analysis(MS-SSCA)</span></a></li>
<li class="toclevel-3"><a href="#d.29_High_resolution_melting_analysis_.28HRM.29"><span class="tocnumber">1.1.4</span> <span class="toctext">d) High resolution melting analysis (HRM)</span></a></li>
<li class="toclevel-3"><a href="#e.29_Methylation-sensitive_single_nucleotide_primer_extension_.28MS-SnuPE.29"><span class="tocnumber">1.1.5</span> <span class="toctext">e) Methylation-sensitive single nucleotide primer extension (MS-SnuPE)</span></a></li>
<li class="toclevel-3"><a href="#f.29_Base-specific_cleavage.2FMALDI-TOF"><span class="tocnumber">1.1.6</span> <span class="toctext">f) Base-specific cleavage/MALDI-TOF</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a href="#2._Methylation-specific_PCR_.28MSP.29"><span class="tocnumber">1.2</span> <span class="toctext">2. Methylation-specific PCR (MSP)</span></a></li>
<li class="toclevel-2"><a href="#3._Microarray-based_methods"><span class="tocnumber">1.3</span> <span class="toctext">3. Microarray-based methods</span></a></li>
</ul>
</li>
<li class="toclevel-1"><a href="#Limitations"><span class="tocnumber">2</span> <span class="toctext">Limitations</span></a>
<ul>
<li class="toclevel-2"><a href="#Incomplete_Conversion:"><span class="tocnumber">2.1</span> <span class="toctext">Incomplete Conversion:</span></a></li>
<li class="toclevel-2"><a href="#Degradation_of_DNA_during_Bisulphite_Treatment:"><span class="tocnumber">2.2</span> <span class="toctext">Degradation of DNA during Bisulphite Treatment:</span></a></li>
<li class="toclevel-2"><a href="#Other_Concerns"><span class="tocnumber">2.3</span> <span class="toctext">Other Concerns</span></a></li>
</ul>
</li>
<li class="toclevel-1"><a href="#Applications:_Genome-Wide_Methylation_Analysis"><span class="tocnumber">3</span> <span class="toctext">Applications: Genome-Wide Methylation Analysis</span></a></li>
<li class="toclevel-1"><a href="#External_Links"><span class="tocnumber">4</span> <span class="toctext">External Links</span></a></li>
<li class="toclevel-1"><a href="#References"><span class="tocnumber">5</span> <span class="toctext">References</span></a></li>
</ul>
</td>
</tr>
</table>
<p><script type="text/javascript">
//<![CDATA[
 if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } 
//]]>
</script><a name="Methods" id="Methods"></a></p>
<h2><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: Methods">edit</a>]</span> <span class="mw-headline">Methods</span></h2>
<p>Bisulfite sequencing applies routine <a href="../../../d/n/a/DNA_sequencing_a448.html" title="DNA sequencing">sequencing</a> methods on bisulfite-treated genomic DNA to determine methylation status at CpG dinucleotides. Other non-sequencing strategies are also employed to interrogate the methylation at specific loci or at a <a href="../../../g/e/n/Genome.html" title="Genome">genome</a>-wide level. All strategies assume complete conversion of unmethylated cytosines to uracil, and this serves as the basis of all subsequent techniques. Ideally, the method used would determine the methylation status separately for each <a href="../../../a/l/l/Allele.html" title="Allele">allele</a>. Alternative methods to bisulfite sequencing include methylation-specific restriction analysis and methylated DNA immunoprecipitation (MeDIP).</p>
<p>Methodologies to analyze bisulfite-treated DNA are continuously being developed. To summarize these rapidly evolving methologies, numerous review articles have been written.<sup id="_ref-0" class="reference"><a href="#_note-0" title="">[1]</a></sup><sup id="_ref-1" class="reference"><a href="#_note-1" title="">[2]</a></sup><sup id="_ref-2" class="reference"><a href="#_note-2" title="">[3]</a></sup><sup id="_ref-3" class="reference"><a href="#_note-3" title="">[4]</a></sup></p>
<p>The methodologies can be generally divided into strategies based on methylation-specific PCR (MSP) (Figure 3), and strategies employing PCR performed under non-methylation-specific conditions (Figure 2). Microarray-based methods use PCR based on non-methylation-specific conditions also.</p>
<p><a name="1._Non-methylation-specific_PCR_based_methods"></a></p>
<h3><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: 1. Non-methylation-specific PCR based methods">edit</a>]</span> <span class="mw-headline">1. Non-methylation-specific PCR based methods</span></h3>
<div class="thumb tright">
<div class="thumbinner" style="width:452px;"><a href="../../../w/i/k/Image%7EWiki_Bisulfite_sequencing_Figure_2_small.png_9b95.html" class="internal" title="Figure 2: DNA methylation analysis methods not based on methylation-specific PCR. Following bisulfite conversion, the genomic DNA is amplified with PCR that does not discriminate between methylated and non-methylated sequences.  The numerous methods available are then used to make the discrimination based on the changes within the amplicon as a result of bisulfite conversion."><img src="../../../upload/thumb/2/24/Wiki_Bisulfite_sequencing_Figure_2_small.png/450px-Wiki_Bisulfite_sequencing_Figure_2_small.png" alt="Figure 2: DNA methylation analysis methods not based on methylation-specific PCR. Following bisulfite conversion, the genomic DNA is amplified with PCR that does not discriminate between methylated and non-methylated sequences.  The numerous methods available are then used to make the discrimination based on the changes within the amplicon as a result of bisulfite conversion." width="450" height="213" longdesc="../../../w/i/k/Image%7EWiki_Bisulfite_sequencing_Figure_2_small.png_9b95.html" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="../../../w/i/k/Image%7EWiki_Bisulfite_sequencing_Figure_2_small.png_9b95.html" class="internal" title="Enlarge"><img src="../../../skins/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
Figure 2: DNA methylation analysis methods not based on methylation-specific PCR. Following bisulfite conversion, the genomic DNA is amplified with PCR that does not discriminate between methylated and non-methylated sequences. The numerous methods available are then used to make the discrimination based on the changes within the amplicon as a result of bisulfite conversion.</div>
</div>
</div>
<p><a name="a.29_Direct_sequencing" id="a.29_Direct_sequencing"></a></p>
<h4><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: a) Direct sequencing">edit</a>]</span> <span class="mw-headline">a) Direct sequencing</span></h4>
<p>The first reported method of methylation analysis using bisulfite-treated DNA utilized <a href="../../../p/c/r/PCR_c0ff.html" title="PCR">PCR</a> and standard dideoxynucleotide <a href="../../../d/n/a/DNA_sequencing_a448.html" title="DNA sequencing">DNA sequencing</a> to directly determine the nucleotides resistant to bisulfite conversion.<sup id="_ref-4" class="reference"><a href="#_note-4" title="">[5]</a></sup> Primers are designed to be strand-specific as well as bisulfite-specific (i.e. primers containing non-CpG cytosines such that they are not complimentary to non-bisulfite-treated DNA), flanking (but not involving) the <a href="../../../m/e/t/Methylation.html" title="Methylation">methylation</a> site of interest. Therefore, it will amplify both methylated and unmethylated sequences, in contrast to methylation-specific <a href="../../../p/c/r/PCR_c0ff.html" title="PCR">PCR</a>. All sites of unmethylated <a href="../../../c/y/t/Cytosine.html" title="Cytosine">cytosines</a> are displayed as <a href="../../../t/h/y/Thymine.html" title="Thymine">thymines</a> in the resulting amplified sequence of the sense strand, and as <a href="../../../a/d/e/Adenine.html" title="Adenine">adenines</a> in the amplified <a href="../../../a/n/t/Antisense.html" title="Antisense">antisense</a> strand. This technique required <a href="../../../c/l/o/Cloning.html" title="Cloning">cloning</a> of the <a href="../../../p/c/r/PCR_c0ff.html" title="PCR">PCR</a> product prior to sequencing for adequate sensitivity, and therefore was a very labour-intensive method unsuitable for higher throughput. Alternatively, <a href="../../../n/e/s/Nested_PCR_cb86.html" title="Nested PCR">nested PCR</a> methods can be used to enhance the product for <a href="../../../s/e/q/Sequencing.html" title="Sequencing">sequencing</a>.</p>
<p>All subsequent <a href="../../../d/n/a/DNA_methylation_b466.html" title="DNA methylation">DNA methylation</a> analysis techniques using bisulfite-treated DNA is based on this report by Frommer et al (Figure 2). Although most other modalities are not true sequencing-based techniques, the term "bisulfite sequencing" is often used to describe bisulfite-conversion <a href="../../../d/n/a/DNA_methylation_b466.html" title="DNA methylation">DNA methylation</a> analysis techniques in general.</p>
<p><a name="b.29_Pyrosequencing" id="b.29_Pyrosequencing"></a></p>
<h4><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: b) Pyrosequencing">edit</a>]</span> <span class="mw-headline">b) Pyrosequencing</span></h4>
<p><a href="../../../p/y/r/Pyrosequencing.html" title="Pyrosequencing">Pyrosequencing</a> has also been used to analyze bisulfite-treated DNA without using methylation-specific <a href="../../../p/c/r/PCR_c0ff.html" title="PCR">PCR</a>.<sup id="_ref-5" class="reference"><a href="#_note-5" title="">[6]</a></sup><sup id="_ref-6" class="reference"><a href="#_note-6" title="">[7]</a></sup> Following <a href="../../../p/c/r/PCR_c0ff.html" title="PCR">PCR</a> amplification of the region of interest, <a href="../../../p/y/r/Pyrosequencing.html" title="Pyrosequencing">Pyrosequencing</a> is used to determine the bisulfite-converted sequence of specific <a href="../../../c/p/g/CpG_site_13ac.html" title="CpG site">CpG sites</a> in the region. The ratio of C-to-T at individual sites can be determined quantitatively based on the amount of C and T incorporation during the sequence extension. The main limitation of this method is the cost of the technology. However, <a href="../../../p/y/r/Pyrosequencing.html" title="Pyrosequencing">Pyrosequencing</a> does well allow for extension to <a href="../../../h/i/g/High-throughput_screening.html" title="High-throughput screening">high-throughput screening</a> methods.</p>
<p>A further improvement to this technique was recently described by Wong et al., which uses allele-specific primers that incorporate <a href="../../../s/i/n/Single-nucleotide_polymorphism.html" title="Single-nucleotide polymorphism">single-nucleotide polymorphisms</a> into the sequence of the sequencing primer, thus allowing for separate analysis of maternal and paternal <a href="../../../a/l/l/Allele.html" title="Allele">alleles</a>.<sup id="_ref-7" class="reference"><a href="#_note-7" title="">[8]</a></sup> This technique is of particular usefulness for <a href="../../../g/e/n/Genomic_imprinting.html" title="Genomic imprinting">genomic imprinting</a> analysis.</p>
<p><a name="c.29_Methylation-sensitive_single-strand_conformation_analysis.28MS-SSCA.29" id="c.29_Methylation-sensitive_single-strand_conformation_analysis.28MS-SSCA.29"></a></p>
<h4><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: c) Methylation-sensitive single-strand conformation analysis(MS-SSCA)">edit</a>]</span> <span class="mw-headline">c) Methylation-sensitive single-strand conformation analysis(MS-SSCA)</span></h4>
<p>This method is based on the <a href="../../../s/i/n/Single_strand_conformation_polymorphism.html" title="Single strand conformation polymorphism">single strand conformation polymorphism</a> analysis (SSCA) method developed for <a href="../../../s/i/n/Single-nucleotide_polymorphism.html" title="Single-nucleotide polymorphism">single-nucleotide polymorphism</a> (SNP) analysis.<sup id="_ref-8" class="reference"><a href="#_note-8" title="">[9]</a></sup> SSCA differentiates between single-stranded DNA fragments of identical size but distinct sequence based on differential migration in non-denaturating <a href="../../../e/l/e/Electrophoresis.html" title="Electrophoresis">electrophoresis</a>. In MS-SSCA, this is used to distinguish between bisulfite-treated, <a href="../../../p/c/r/PCR_c0ff.html" title="PCR">PCR</a>-amplified regions containing the CpG sites of interest. Although SSCA lacks sensitivity when only a single <a href="../../../n/u/c/Nucleotide.html" title="Nucleotide">nucleotide</a> difference is present, <a href="../../../b/i/s/Bisulfite.html" title="Bisulfite">bisulfite</a> treatment frequently makes a number of C-to-T conversions in most regions of interest, and the resulting sensitivity approaches 100%. MS-SSCA also provides semi-quantitative analysis of the degree of <a href="../../../d/n/a/DNA_methylation_b466.html" title="DNA methylation">DNA methylation</a> based on the ratio of band intensities. However, this method is ideally designed to assess all <a href="../../../c/p/g/CpG_site_13ac.html" title="CpG site">CpG sites</a> as a whole in the region of interest rather than individual <a href="../../../m/e/t/Methylation.html" title="Methylation">methylation</a> sites.</p>
<p><a name="d.29_High_resolution_melting_analysis_.28HRM.29" id="d.29_High_resolution_melting_analysis_.28HRM.29"></a></p>
<h4><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: d) High resolution melting analysis (HRM)">edit</a>]</span> <span class="mw-headline">d) High resolution melting analysis (HRM)</span></h4>
<p>A further method to differentiate converted from unconverted bisulfite-treated DNA is using high resolution melting analysis (HRM), a <a href="../../../r/e/a/Real-time_PCR_c4f8.html" title="Real-time PCR">real-time PCR</a>-based technique initially designed to distinguish <a href="../../../s/n/p/SNP_4fc6.html" title="SNP">SNPs</a>.<sup id="_ref-9" class="reference"><a href="#_note-9" title="">[10]</a></sup> The <a href="../../../p/c/r/PCR_c0ff.html" title="PCR">PCR</a> <a href="../../../a/m/p/Amplicons.html" title="Amplicons">amplicons</a> are analyzed directly by temperature ramping and resulting liberation of an intercalating <a href="../../../f/l/u/Fluorescent_dye.html" title="Fluorescent dye">fluorescent dye</a> during melting. The degree of methylation, as represented by the C-to-T content in the <a href="../../../a/m/p/Amplicon.html" title="Amplicon">amplicon</a>, determines the rapidity of melting and consequent release of the dye. This method allows direct quantitation in a single-tube assay, but again assesses <a href="../../../m/e/t/Methylation.html" title="Methylation">methylation</a> in the amplified region as a whole rather than at specific <a href="../../../c/p/g/CpG_site_13ac.html" title="CpG site">CpG sites</a>.</p>
<p><a name="e.29_Methylation-sensitive_single_nucleotide_primer_extension_.28MS-SnuPE.29" id="e.29_Methylation-sensitive_single_nucleotide_primer_extension_.28MS-SnuPE.29"></a></p>
<h4><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: e) Methylation-sensitive single nucleotide primer extension (MS-SnuPE)">edit</a>]</span> <span class="mw-headline">e) Methylation-sensitive single nucleotide primer extension (MS-SnuPE)</span></h4>
<p>MS-SnuPE employs the primer extension method initially designed for analyzing <a href="../../../s/i/n/Single-nucleotide_polymorphism.html" title="Single-nucleotide polymorphism">single-nucleotide polymorphisms</a>.<sup id="_ref-10" class="reference"><a href="#_note-10" title="">[11]</a></sup> DNA is bisulfite-converted, and bisulfite-specific primers are annealed to the sequence up to the base pair immediately before the CpG of interest. The primer is allowed to extend one base pair into the C (or T) using <a href="../../../d/n/a/DNA_polymerase_7877.html" title="DNA polymerase">DNA polymerase</a> terminating <a href="../../../d/i/d/Dideoxynucleotides.html" title="Dideoxynucleotides">dideoxynucleotides</a>, and the ratio of C to T is determined quantitatively.</p>
<p>A number of methods can be used to determine this C:T ratio. Initially, MS-SnuPE relied on radioactive <a href="../../../d/d/n/DdNTP_bf88.html" title="DdNTP">ddNTPs</a> as the reporter of the primer extension. Fluorescence-based methods or <a href="../../../p/y/r/Pyrosequencing.html" title="Pyrosequencing">Pyrosequencing</a> can also be used.<sup id="_ref-11" class="reference"><a href="#_note-11" title="">[12]</a></sup> However, matrix-assisted laser desorption ionization/time-of-flight (<a href="../../../m/a/l/MALDI-TOF_274d.html" title="MALDI-TOF">MALDI-TOF</a>) <a href="../../../m/a/s/Mass_spectrometry.html" title="Mass spectrometry">mass spectrometry</a> analysis to differentiate between the two polymorphic primer extension products can be used, essentially based on the GOOD assay designed for <a href="../../../s/n/p/SNP_genotyping_dd30.html" title="SNP genotyping">SNP genotyping</a>. Ion pair reverse-phase <a href="../../../h/i/g/High_performance_liquid_chromatography.html" title="High performance liquid chromatography">high performance liquid chromatography</a> (IP-RP-<a href="../../../h/p/l/HPLC_40a4.html" title="HPLC">HPLC</a>) has also been used to distinguish primer extension products.<sup id="_ref-12" class="reference"><a href="#_note-12" title="">[13]</a></sup></p>
<p><a name="f.29_Base-specific_cleavage.2FMALDI-TOF" id="f.29_Base-specific_cleavage.2FMALDI-TOF"></a></p>
<h4><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: f) Base-specific cleavage/MALDI-TOF">edit</a>]</span> <span class="mw-headline">f) Base-specific cleavage/MALDI-TOF</span></h4>
<p>A recently described method by Ehrich et al. further takes advantage of bisulfite-conversions by adding a base-specific cleavage step to enhance the information gained from the nucleotide changes.<sup id="_ref-13" class="reference"><a href="#_note-13" title="">[14]</a></sup> By first using in vitro <a href="../../../t/r/a/Transcription_%28genetics%29.html" title="Transcription (genetics)">transcription</a> of the region of interest into <a href="../../../r/n/a/RNA_78c4.html" title="RNA">RNA</a> (by adding an <a href="../../../r/n/a/RNA_polymerase_075b.html" title="RNA polymerase">RNA polymerase</a> <a href="../../../p/r/o/Promoter.html" title="Promoter">promoter</a> site to the <a href="../../../p/c/r/PCR_c0ff.html" title="PCR">PCR</a> primer in the initial amplification), <a href="../../../r/n/a/RNase_A_eed6.html" title="RNase A">RNase A</a> can be used to cleave the <a href="../../../r/n/a/RNA_78c4.html" title="RNA">RNA</a> <a href="../../../t/r/a/Transcript.html" title="Transcript">transcript</a> at base-specific sites. As <a href="../../../r/n/a/RNase_A_eed6.html" title="RNase A">RNase A</a> cleaves <a href="../../../r/n/a/RNA_78c4.html" title="RNA">RNA</a> specifically at <a href="../../../c/y/t/Cytosine.html" title="Cytosine">cytosine</a> and <a href="../../../u/r/a/Uracil.html" title="Uracil">uracil</a> <a href="../../../r/i/b/Ribonucleotides.html" title="Ribonucleotides">ribonucleotides</a>, base-specificity is achieved by adding incorporating cleavage-resistant dTTP when <a href="../../../c/y/t/Cytosine.html" title="Cytosine">cytosine</a>-specific (C-specific) cleavage is desired, and incorporating dCTP when uracil-specific (U-specific) cleavage is desired. The cleaved fragments can then be analyzed by <a href="../../../m/a/l/MALDI-TOF_274d.html" title="MALDI-TOF">MALDI-TOF</a>. Bisulfite treatment either results in introduction/removal of cleavage sites by C-to-U conversions, or shift in fragment mass by G-to-A (<a href="../../../p/u/r/Purine.html" title="Purine">purine</a>-to-<a href="../../../p/y/r/Pyrimidine.html" title="Pyrimidine">pyrimidine</a>) conversions in the amplified reverse strand. Most informatively, C-specific cleavage will cut specifically at all methylated <a href="../../../c/p/g/CpG_site_13ac.html" title="CpG site">CpG sites</a>. By analyzing the sizes of the resulting fragments, it is possible to determine the specific pattern of <a href="../../../d/n/a/DNA_methylation_b466.html" title="DNA methylation">DNA methylation</a> of <a href="../../../c/p/g/CpG_site_13ac.html" title="CpG site">CpG sites</a> with the region, rather than determining the extent of <a href="../../../m/e/t/Methylation.html" title="Methylation">methylation</a> of the region as a whole. This method demonstrated efficacy for <a href="../../../h/i/g/High-throughput_screening.html" title="High-throughput screening">high-throughput screening</a>, allowing for interrogation of numerous <a href="../../../c/p/g/CpG_site_13ac.html" title="CpG site">CpG sites</a> in multiple tissues in a cost-efficient manner.</p>
<p><a name="2._Methylation-specific_PCR_.28MSP.29"></a></p>
<h3><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: 2. Methylation-specific PCR (MSP)">edit</a>]</span> <span class="mw-headline">2. Methylation-specific PCR (MSP)</span></h3>
<div class="thumb tright">
<div class="thumbinner" style="width:402px;"><a href="../../../w/i/k/Image%7EWiki_Bisulfite_sequencing_Figure_3_small.png_9804.html" class="internal" title="Figure 3: Methylation-specific PCR is a sensitive method to discriminately amplify and detect a methylated region of interest using methylated-specific primers on bisulfite-converted genomic DNA.  Such primers will only anneal to sequences that are methylated, and thus containing 5-methylcytosines that are resistant to conversion by bisulfite.  Alternatively, unmethylated-specific primers can be used."><img src="../../../upload/thumb/2/24/Wiki_Bisulfite_sequencing_Figure_3_small.png/400px-Wiki_Bisulfite_sequencing_Figure_3_small.png" alt="Figure 3: Methylation-specific PCR is a sensitive method to discriminately amplify and detect a methylated region of interest using methylated-specific primers on bisulfite-converted genomic DNA.  Such primers will only anneal to sequences that are methylated, and thus containing 5-methylcytosines that are resistant to conversion by bisulfite.  Alternatively, unmethylated-specific primers can be used." width="400" height="238" longdesc="../../../w/i/k/Image%7EWiki_Bisulfite_sequencing_Figure_3_small.png_9804.html" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="../../../w/i/k/Image%7EWiki_Bisulfite_sequencing_Figure_3_small.png_9804.html" class="internal" title="Enlarge"><img src="../../../skins/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
Figure 3: Methylation-specific <a href="../../../p/c/r/PCR_c0ff.html" title="PCR">PCR</a> is a sensitive method to discriminately amplify and detect a methylated region of interest using methylated-specific primers on bisulfite-converted genomic DNA. Such primers will only anneal to sequences that are methylated, and thus containing <a href="../../../5/-/m/5-methylcytosine.html" title="5-methylcytosine">5-methylcytosines</a> that are resistant to conversion by bisulfite. Alternatively, unmethylated-specific primers can be used.</div>
</div>
</div>
<p>This alternative method of methylation analysis also uses bisulfite-treated DNA but avoids the need to sequence the area of interest.<sup id="_ref-14" class="reference"><a href="#_note-14" title="">[15]</a></sup> Instead, primer pairs are designed themselves to be "methylated-specific" by including sequences complementing only unconverted <a href="../../../5/-/m/5-methylcytosine.html" title="5-methylcytosine">5-methylcytosines</a>, or conversely "unmethylated-specific", complementing <a href="../../../t/h/y/Thymine.html" title="Thymine">thymines</a> converted from unmethylated <a href="../../../c/y/t/Cytosine.html" title="Cytosine">cytosines</a>. <a href="../../../m/e/t/Methylation.html" title="Methylation">Methylation</a> is determined by the ability of the specific primer to achieve amplification. This method is particularly useful to interrogate <a href="../../../c/p/g/CpG_islands_1e20.html" title="CpG islands">CpG islands</a> with possibly high methylation density, as increased numbers of CpG pairs in the primer increase the specificity of the assay. Placing the CpG pair at the 3'-end of the primer also improves the sensitivity. The initial report using MSP described sufficient sensitivity to detect methylation of 0.1% of <a href="../../../a/l/l/Allele.html" title="Allele">alleles</a>. In general, MSP and its related protocols are considered to be the most sensitive when interrogating the methylation status at a specific <a href="../../../l/o/c/Locus.html" title="Locus">locus</a>.</p>
<p>The MethyLight method is based on MSP, but provides a quantitative analysis using <a href="../../../r/e/a/Real-time_PCR_c4f8.html" title="Real-time PCR">real-time PCR</a>.<sup id="_ref-15" class="reference"><a href="#_note-15" title="">[16]</a></sup> Methylated-specific primers are used, and a methylated-specific fluorescence reporter probe is also used that anneals to the amplified region. Alternatively, the primers or probe can be designed without methylation specificity if discrimination is needed between the CpG pairs within the involved sequences. Quantitation is made in reference to a methylated reference DNA. A modification to this protocol to increase the specificity of the PCR for successfully bisulfite-converted DNA (ConLight-MSP) uses an additional probe to bisulfite-unconverted DNA to quantify this non-specific amplification.<sup id="_ref-16" class="reference"><a href="#_note-16" title="">[17]</a></sup></p>
<p>A further method using MSP-amplified DNA analyzes the products using melting-curve analysis (Mc-MSP).<sup id="_ref-17" class="reference"><a href="#_note-17" title="">[18]</a></sup> This method amplifies bisulfite-converted DNA with both methylated-specific and unmethylated-specific primers, and determines the quantitative ratio of the two products by comparing the differential peaks generated in a melting-curve analysis.</p>
<p><a name="3._Microarray-based_methods"></a></p>
<h3><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: 3. Microarray-based methods">edit</a>]</span> <span class="mw-headline">3. Microarray-based methods</span></h3>
<p><a href="../../../m/i/c/Microarray.html" title="Microarray">Microarray</a>-based methods are a logical extension of the technologies available to analyze bisulfite-treated DNA to allow for genome-wide analysis of methylation.<sup id="_ref-18" class="reference"><a href="#_note-18" title="">[19]</a></sup> <a href="../../../o/l/i/Oligonucleotide_microarray.html" title="Oligonucleotide microarray">Oligonucleotide microarrays</a> are designed using <a href="../../../o/l/i/Oligonucleotide.html" title="Oligonucleotide">oligonucleotide</a> pairs targeting <a href="../../../c/p/g/CpG_site_13ac.html" title="CpG site">CpG sites</a> of interest, with one complementary to the unaltered methylated sequence, and the other to the C-to-T converted unmethylated sequence. The <a href="../../../o/l/i/Oligonucleotide.html" title="Oligonucleotide">oligonucleotides</a> are also bisulfite-specific to prevent binding to DNA incompletely converted by bisulfite.</p>
<p><a name="Limitations" id="Limitations"></a></p>
<h2><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: Limitations">edit</a>]</span> <span class="mw-headline">Limitations</span></h2>
<p><a name="Incomplete_Conversion:" id="Incomplete_Conversion:"></a></p>
<h4><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: Incomplete Conversion:">edit</a>]</span> <span class="mw-headline">Incomplete Conversion:</span></h4>
<p>Bisulphite sequencing relies on the conversion of every single unmethylated <a href="../../../c/y/t/Cytosine.html" title="Cytosine">cytosine</a> residue to <a href="../../../u/r/a/Uracil.html" title="Uracil">uracil</a>. If conversion is incomplete, the subsequent analysis will incorrectly interpret the unconverted unmethylated cytosines as methylated cytosines, resulting in <a href="../../../f/a/l/False_positive.html" title="False positive">false positive</a> results for methylation. Only cytosines in single-stranded DNA are susceptible to attack by bisulphite, therefore <a href="../../../d/e/n/Denaturation.html" title="Denaturation">denaturation</a> of the DNA undergoing analysis is critical.<sup id="_ref-19" class="reference"><a href="#_note-19" title="">[20]</a></sup> It is important to ensure that reaction parameters such as temperature and salt concentration are suitable to maintain the DNA in a single-stranded conformation and allow for complete conversion. Embedding the DNA in <a href="../../../a/g/a/Agarose.html" title="Agarose">agarose</a> gel has been reported to improve the rate of conversion by keeping strands of DNA physically separate.<sup id="_ref-20" class="reference"><a href="#_note-20" title="">[21]</a></sup></p>
<p><a name="Degradation_of_DNA_during_Bisulphite_Treatment:" id="Degradation_of_DNA_during_Bisulphite_Treatment:"></a></p>
<h4><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: Degradation of DNA during Bisulphite Treatment:">edit</a>]</span> <span class="mw-headline">Degradation of DNA during Bisulphite Treatment:</span></h4>
<p>A major challenge in bisulphite sequencing is the degradation of DNA that takes place concurrently with the conversion. The conditions necessary for complete conversion, such as long incubation times, elevated temperature, and high bisulphite concentration, can lead to the degradation of about 90% of the incubated DNA.<sup id="_ref-21" class="reference"><a href="#_note-21" title="">[22]</a></sup> Given that the starting amount of DNA is often limited, such extensive degradation can be problematic. The degradation occurs as <a href="../../../d/e/p/Depurination.html" title="Depurination">depurinations</a> resulting in random strand breaks.<sup id="_ref-22" class="reference"><a href="#_note-22" title="">[23]</a></sup> Therefore the longer the desired <a href="../../../p/c/r/PCR_c0ff.html" title="PCR">PCR</a> <a href="../../../a/m/p/Amplicon.html" title="Amplicon">amplicon</a>, the more limited the number of intact template molecules will likely be. This could lead to the failure of the PCR amplification, or the loss of quantitatively accurate information on methylation levels resulting from the limited <a href="../../../s/a/m/Sampling_%28statistics%29.html" title="Sampling (statistics)">sampling</a> of template molecules. It is thus important to assess the amount of DNA degradation resulting from the reaction conditions employed, and consider how this will affect the desired <a href="../../../a/m/p/Amplicon.html" title="Amplicon">amplicon</a>. Techniques can also be used to minimize DNA degradation, such as cycling the incubation temperature.<sup id="_ref-23" class="reference"><a href="#_note-23" title="">[24]</a></sup></p>
<p><a name="Other_Concerns" id="Other_Concerns"></a></p>
<h4><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: Other Concerns">edit</a>]</span> <span class="mw-headline">Other Concerns</span></h4>
<p>A potentially significant problem following bisulphite treatment is incomplete <a href="../../../s/u/l/Sulfonic_acid.html" title="Sulfonic acid">desulfonation</a> of <a href="../../../p/y/r/Pyrimidine.html" title="Pyrimidine">pyrimidine</a> residues due to inadequate alkalization of the solution. This may inhibit some <a href="../../../d/n/a/DNA_polymerases_77c5.html" title="DNA polymerases">DNA polymerases</a>, rendering subsequent PCR difficult. However this situation can be avoided by monitoring the <a href="../../../p/h/_/PH_a254.html" title="PH">pH</a> of the solution to ensure that desulphonation will be complete.<sup id="_ref-24" class="reference"><a href="#_note-24" title="">[25]</a></sup></p>
<p>A final concern is that bisulphite treatment greatly reduces the level of complexity in the sample, which can be problematic if multiple PCR reactions are to be performed (2006).<sup id="_ref-25" class="reference"><a href="#_note-25" title="">[26]</a></sup> <a href="../../../p/r/i/Primer_%28molecular_biology%29.html" title="Primer (molecular biology)">Primer</a> design is more difficult, and inappropriate cross-hybridization is more frequent.</p>
<p><a name="Applications:_Genome-Wide_Methylation_Analysis" id="Applications:_Genome-Wide_Methylation_Analysis"></a></p>
<h2><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: Applications: Genome-Wide Methylation Analysis">edit</a>]</span> <span class="mw-headline">Applications: Genome-Wide Methylation Analysis</span></h2>
<p>The advances in bisulfite sequencing have led to the possibility of applying them at a <a href="../../../g/e/n/Genome.html" title="Genome">genome</a>-wide scale. The mapping of the human <a href="../../../e/p/i/Epigenome.html" title="Epigenome">epigenome</a> is seen by many scientists as the logical follow-up to the completion of the <a href="../../../h/u/m/Human_Genome_Project_c567.html" title="Human Genome Project">Human Genome Project</a>.<sup id="_ref-26" class="reference"><a href="#_note-26" title="">[27]</a></sup><sup id="_ref-27" class="reference"><a href="#_note-27" title="">[28]</a></sup> This epigenomic information will be important in understanding how the function of the genetic sequence is implemented and regulated. Since the epigenome is less stable than the genome, it is thought to be important in <a href="../../../g/e/n/Gene-environment_interaction.html" title="Gene-environment interaction">gene-environment interactions</a>.<sup id="_ref-28" class="reference"><a href="#_note-28" title="">[29]</a></sup></p>
<p>Epigenomic mapping is inherently more complex than <a href="../../../g/e/n/Genome_sequencing.html" title="Genome sequencing">genome sequencing</a>, however, since the <a href="../../../e/p/i/Epigenome.html" title="Epigenome">epigenome</a> is much more variable than the <a href="../../../g/e/n/Genome.html" title="Genome">genome</a>. While an individual only has one genome, one’s epigenome varies with age, differs between tissues, is altered by environmental factors, and shows aberrations in diseases. Such rich epigenomic mapping however, representing different ages, tissue types, and disease states, would yield valuable information on the normal function of <a href="../../../e/p/i/Epigenetic.html" title="Epigenetic">epigenetic</a> marks as well as the mechanisms leading to aging and disease.</p>
<p>Direct benefits of epigenomic mapping include probable advances in <a href="../../../c/l/o/Cloning.html" title="Cloning">cloning</a> technology. It is believed that failures to produce cloned animals with normal viability and lifespan result from inappropriate patterns of epigenetic marks. Also, aberrant methylation patterns are well characterized in many <a href="../../../c/a/n/Cancers.html" title="Cancers">cancers</a>. Global hypomethylation results in decreased genomic stability, while local hypermethylation of <a href="../../../t/u/m/Tumour_suppressor_gene.html" title="Tumour suppressor gene">tumour suppressor gene</a> <a href="../../../p/r/o/Promoters.html" title="Promoters">promoters</a> often accounts for their <a href="../../../l/o/s/Loss_of_function.html" title="Loss of function">loss of function</a>. Specific patterns of methylation are indicative of specific cancer types, have <a href="../../../p/r/o/Prognosis.html" title="Prognosis">prognostic</a> value and can help to guide the best course of treatment.<sup id="_ref-29" class="reference"><a href="#_note-29" title="">[30]</a></sup></p>
<p>Large-scale <a href="../../../e/p/i/Epigenome.html" title="Epigenome">epigenome</a> mapping efforts are under way around the world and have been organized under the Human Epigenome Project.<sup id="_ref-30" class="reference"><a href="#_note-30" title="">[31]</a></sup> This is based on a multi-tiered strategy, whereby bisulfite sequencing is used to obtain high resolution methylation profiles for a limited number of reference epigenomes, while less thorough analysis is performed on a wider spectrum of samples. This approach is intended to maximize the insight gained from a given amount of ressources, as high-resolution genome-wide mapping remains a costly undertaking.</p>
<p><a name="External_Links" id="External_Links"></a></p>
<h2><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: External Links">edit</a>]</span> <span class="mw-headline">External Links</span></h2>
<ul>
<li><a href="http://www.epigenome.org/" class="external free" title="http://www.epigenome.org/" rel="nofollow">http://www.epigenome.org/</a> - Human Epigenome Project</li>
<li><a href="http://www.sanger.ac.uk/PostGenomics/epigenome/" class="external free" title="http://www.sanger.ac.uk/PostGenomics/epigenome/" rel="nofollow">http://www.sanger.ac.uk/PostGenomics/epigenome/</a> - Sanger Institute</li>
</ul>
<p><a name="References" id="References"></a></p>
<h2><span class="editsection">[<a href="../../../b/i/s/Bisulfite_sequencing.html" title="Edit section: References">edit</a>]</span> <span class="mw-headline">References</span></h2>
<ol class="references">
<li id="_note-0"><b><a href="#_ref-0" title="">^</a></b> Fraga MF, Esteller M: DNA methylation: a profile of methods and applications. Biotechniques 2002, 33:632, 634, 636-649</li>
<li id="_note-1"><b><a href="#_ref-1" title="">^</a></b> El-Maarri O: Methods: DNA methylation. Adv Exp Med Biol 2003, 544:197-204</li>
<li id="_note-2"><b><a href="#_ref-2" title="">^</a></b> Laird PW: The power and the promise of DNA methylation markers. Nat Rev Cancer 2003, 3:253-266</li>
<li id="_note-3"><b><a href="#_ref-3" title="">^</a></b> Callinan PA, Feinberg AP: The emerging science of epigenomics. Hum Mol Genet 2006, 15 Spec No 1:R95-101</li>
<li id="_note-4"><b><a href="#_ref-4" title="">^</a></b> Frommer M, McDonald LE, Millar DS, Collis CM, Watt F, Grigg GW, Molloy PL, Paul CL: A genomic sequencing protocol that yields a positive display of 5-methylcytosine residues in individual DNA strands. Proc Natl Acad Sci U S A 1992, 89:1827-1831</li>
<li id="_note-5"><b><a href="#_ref-5" title="">^</a></b> Colella S, Shen L, Baggerly KA, Issa JP, Krahe R: Sensitive and quantitative universal Pyrosequencing methylation analysis of CpG sites. Biotechniques 2003, 35:146-150</li>
<li id="_note-6"><b><a href="#_ref-6" title="">^</a></b> Tost J, Dunker J, Gut IG: Analysis and quantification of multiple methylation variable positions in CpG islands by Pyrosequencing. Biotechniques 2003, 35:152-156</li>
<li id="_note-7"><b><a href="#_ref-7" title="">^</a></b> Wong HL, Byun HM, Kwan JM, Campan M, Ingles SA, Laird PW, Yang AS: Rapid and quantitative method of allele-specific DNA methylation analysis. Biotechniques 2006, 41:734-739</li>
<li id="_note-8"><b><a href="#_ref-8" title="">^</a></b> Bianco T, Hussey D, Dobrovic A: Methylation-sensitive, single-strand conformation analysis (MS-SSCA): A rapid method to screen for and analyze methylation. Hum Mutat 1999, 14:289-293</li>
<li id="_note-9"><b><a href="#_ref-9" title="">^</a></b> Wojdacz TK, Dobrovic A: Methylation-sensitive high resolution melting (MS-HRM): a new approach for sensitive and high-throughput assessment of methylation. Nucleic Acids Res 2007</li>
<li id="_note-10"><b><a href="#_ref-10" title="">^</a></b> Gonzalgo ML, Jones PA: Rapid quantitation of methylation differences at specific sites using methylation-sensitive single nucleotide primer extension (Ms-SNuPE). Nucleic Acids Res 1997, 25:2529-2531</li>
<li id="_note-11"><b><a href="#_ref-11" title="">^</a></b> Uhlmann K, Brinckmann A, Toliat MR, Ritter H, Nurnberg P: Evaluation of a potential epigenetic biomarker by quantitative methyl-single nucleotide polymorphism analysis. Electrophoresis 2002, 23:4072-4079</li>
<li id="_note-12"><b><a href="#_ref-12" title="">^</a></b> Matin MM, Baumer A, Hornby DP: An analytical method for the detection of methylation differences at specific chromosomal loci using primer extension and ion pair reverse phase HPLC. Hum Mutat 2002, 20:305-311</li>
<li id="_note-13"><b><a href="#_ref-13" title="">^</a></b> Ehrich M, Nelson MR, Stanssens P, Zabeau M, Liloglou T, Xinarianos G, Cantor CR, Field JK, van den Boom D: Quantitative high-throughput analysis of DNA methylation patterns by base-specific cleavage and mass spectrometry. Proc Natl Acad Sci U S A 2005, 102:15785-15790</li>
<li id="_note-14"><b><a href="#_ref-14" title="">^</a></b> Herman JG, Graff JR, Myohanen S, Nelkin BD, Baylin SB: Methylation-specific PCR: a novel PCR assay for methylation status of CpG islands. Proc Natl Acad Sci U S A 1996, 93:9821-9826</li>
<li id="_note-15"><b><a href="#_ref-15" title="">^</a></b> Eads CA, Danenberg KD, Kawakami K, Saltz LB, Blake C, Shibata D, Danenberg PV, Laird PW: MethyLight: a high-throughput assay to measure DNA methylation. Nucleic Acids Res 2000, 28:E32</li>
<li id="_note-16"><b><a href="#_ref-16" title="">^</a></b> Rand K, Qu W, Ho T, Clark SJ, Molloy P: Conversion-specific detection of DNA methylation using real-time polymerase chain reaction (ConLight-MSP) to avoid false positives. Methods 2002, 27:114-120</li>
<li id="_note-17"><b><a href="#_ref-17" title="">^</a></b> Akey DT, Akey JM, Zhang K, Jin L: Assaying DNA methylation based on high-throughput melting curve approaches. Genomics 2002, 80:376-384</li>
<li id="_note-18"><b><a href="#_ref-18" title="">^</a></b> Adorjan P, Distler J, Lipscher E, Model F, Muller J, Pelet C, Braun A, Florl AR, Gutig D, Grabs G, Howe A, Kursar M, Lesche R, Leu E, Lewin A, Maier S, Muller V, Otto T, Scholz C, Schulz WA, Seifert HH, Schwope I, Ziebarth H, Berlin K, Piepenbrock C, Olek A: Tumour class prediction and discovery by microarray-based DNA methylation analysis. Nucleic Acids Res 2002, 30:E21</li>
<li id="_note-19"><b><a href="#_ref-19" title="">^</a></b> Fraga MF, Esteller M: DNA methylation: A Profile of Methods and Applications. Biotechniques 2002, 33:632-649</li>
<li id="_note-20"><b><a href="#_ref-20" title="">^</a></b> Olek A, Oswald J, Walter J: A modified and improved method for bisulphite based cytosine methylation analysis. Nucleic Acids Res 1996, 24(24):5064-6</li>
<li id="_note-21"><b><a href="#_ref-21" title="">^</a></b> Grunau C, Clark SJ, Rosenthal A: Bisulfite genomic sequencing:systematic investigation o critical experimental parameters. Nucleic Acids Res 2001, 29:E65</li>
<li id="_note-22"><b><a href="#_ref-22" title="">^</a></b> Ehrich M, Zoll S, Sur S, van den Boom D: A new method for accurate assessment of DNA quality after bisulfite treatment. Nucleic Acids Res. 2007</li>
<li id="_note-23"><b><a href="#_ref-23" title="">^</a></b> Ehrich M, Zoll S, Sur S, van den Boom D: A new method for accurate assessment of DNA quality after bisulfite treatment. Nucleic Acids Res. 2007</li>
<li id="_note-24"><b><a href="#_ref-24" title="">^</a></b> Fraga MF, Esteller M: DNA methylation: A Profile of Methods and Applications. Biotechniques 2002, 33:632-649</li>
<li id="_note-25"><b><a href="#_ref-25" title="">^</a></b> Callinan PA, Feinberg AP: The emerging science of epigenomics. Hum Mol Genet 2006, 15:R95-101</li>
<li id="_note-26"><b><a href="#_ref-26" title="">^</a></b> Esteller M: The necessity of a human epigenome project. Carcinogenesis 2006, 27:1121-5</li>
<li id="_note-27"><b><a href="#_ref-27" title="">^</a></b> Bradbury M: Human Epigenome Project-Up and Running. PLoS Biology 2003, 1:E82</li>
<li id="_note-28"><b><a href="#_ref-28" title="">^</a></b> Jones P, Martienssen R: A Blueprint for a Human Epigenome Project: The AACR Human Epigenome Workshop. Cancer Res 2005, 65:11241-6</li>
<li id="_note-29"><b><a href="#_ref-29" title="">^</a></b> Bradbury M: Human Epigenome Project-Up and Running. PLoS Biology 2003, 1:E82</li>
<li id="_note-30"><b><a href="#_ref-30" title="">^</a></b> Jones P, Martienssen R: A Blueprint for a Human Epigenome Project: The AACR Human Epigenome Workshop. Cancer Res 2005, 65:11241-6</li>
</ol>

<div class="printfooter">
Retrieved from "<a href="http://en.wikipedia.org../../../b/i/s/Bisulfite_sequencing.html">http://en.wikipedia.org../../../b/i/s/Bisulfite_sequencing.html</a>"</div>
	    <div id="catlinks"><p class='catlinks'><a href="../../../c/a/t/Special%7ECategories_101d.html" title="Special:Categories">Categories</a>: <span dir='ltr'><a href="../../../m/o/l/Category%7EMolecular_biology_6327.html" title="Category:Molecular biology">Molecular biology</a></span> | <span dir='ltr'><a href="../../../e/p/i/Category%7EEpigenetics_d483.html" title="Category:Epigenetics">Epigenetics</a></span> | <span dir='ltr'><a href="../../../g/e/n/Category%7EGenomics_d747.html" title="Category:Genomics">Genomics</a></span></p></div>	    <!-- end content -->
	    <div class="visualClear"></div>
	  </div>
	</div>
      </div>
      <div id="column-one">
	<div id="p-cactions" class="portlet">
	  <h5>Views</h5>
	  <ul>
	    <li id="ca-nstab-main"
	       class="selected"	       ><a href="../../../b/i/s/Bisulfite_sequencing.html">Article</a></li><li id="ca-talk"
	       class="new"	       ><a href="../../../b/i/s/Talk%7EBisulfite_sequencing_3d2f.html">Discussion</a></li><li id="ca-current"
	       	       ><a href="http://en.wikipedia.org/wiki/Bisulfite_sequencing">Current revision</a></li>	  </ul>
	</div>
	<div class="portlet" id="p-logo">
	  <a style="background-image: url(../../../images/wiki-en.png);"
	    href="../../../index.html"
	    title="Main Page"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
		<div class='portlet' id='p-navigation'>
	  <h5>Navigation</h5>
	  <div class='pBody'>
	    <ul>
	    	      <li id="n-Main-page"><a href="../../../index.html">Main page</a></li>
	     	      <li id="n-Contents"><a href="../../../c/o/n/Wikipedia%7EContents_3181.html">Contents</a></li>
	     	      <li id="n-Featured-content"><a href="../../../f/e/a/Wikipedia%7EFeatured_content_24ba.html">Featured content</a></li>
	     	      <li id="n-currentevents"><a href="../../../c/u/r/Portal%7ECurrent_events_bb60.html">Current events</a></li>
	     	    </ul>
	  </div>
	</div>
		<div class='portlet' id='p-interaction'>
	  <h5>interaction</h5>
	  <div class='pBody'>
	    <ul>
	    	      <li id="n-About-Wikipedia"><a href="../../../a/b/o/Wikipedia%7EAbout_8d82.html">About Wikipedia</a></li>
	     	      <li id="n-portal"><a href="../../../c/o/m/Wikipedia%7ECommunity_Portal_6a3c.html">Community portal</a></li>
	     	      <li id="n-contact"><a href="../../../c/o/n/Wikipedia%7EContact_us_afd6.html">Contact us</a></li>
	     	      <li id="n-sitesupport"><a href="http://wikimediafoundation.org/wiki/Fundraising">Make a donation</a></li>
	     	      <li id="n-help"><a href="../../../c/o/n/Help%7EContents_22de.html">Help</a></li>
	     	    </ul>
	  </div>
	</div>
		<div id="p-search" class="portlet">
	  <h5><label for="searchInput">Search</label></h5>
	  <div id="searchBody" class="pBody">
	    <form action="javascript:goToStatic(3)" id="searchform"><div>
	      <input id="searchInput" name="search" type="text"
	        accesskey="f" value="" />
	      <input type='submit' name="go" class="searchButton" id="searchGoButton"
	        value="Go" />
	    </div></form>
	  </div>
	</div>
	      </div><!-- end of the left (by default at least) column -->
      <div class="visualClear"></div>
      <div id="footer">
    <div id="f-poweredbyico"><a href="http://www.mediawiki.org/"><img src="../../../skins/common/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" /></a></div>	<div id="f-copyrightico"><a href="http://wikimediafoundation.org/"><img src="../../../images/wikimedia-button.png" border="0" alt="Wikimedia Foundation"/></a></div>	<ul id="f-list">
	  	  	  <li id="f-credits">This page was last modified 17:22, 2 March 2007 by Wikipedia user Wikipolonius. Based on work by Wikipedia user(s) Toeng and <a href="../../../d/e/k/User%7EDekimasu_4909.html" title="User:Dekimasu">Dekimasu</a> and Anonymous user(s) of Wikipedia.</li>	  <li id="f-copyright">All text is available under the terms of the <a class='internal' href="../../../t/e/x/Wikipedia%7EText_of_the_GNU_Free_Documentation_License_702a.html" title="Wikipedia:Text of the GNU Free Documentation License">GNU Free Documentation License</a>. (See <b><a class='internal' href="../../../c/o/p/Wikipedia%7ECopyrights_92c4.html" title="Wikipedia:Copyrights">Copyrights</a></b> for details.) <br /> Wikipedia&reg; is a registered trademark of the <a href="http://www.wikimediafoundation.org">Wikimedia Foundation, Inc</a>., a US-registered <a class='internal' href="../../../5/0/1/501%28c%29.html#501.28c.29.283.29" title="501(c)(3)">501(c)(3)</a> <a href="http://wikimediafoundation.org/wiki/Deductibility_of_donations">tax-deductible</a> <a class='internal' href="../../../n/o/n/Non-profit_organization.html" title="Non-profit organization">nonprofit</a> <a href="../../../c/h/a/Charitable_organization.html" title="Charitable organization">charity</a>.<br /></li>	  <li id="f-about"><a href="../../../a/b/o/Wikipedia%7EAbout_8d82.html" title="Wikipedia:About">About Wikipedia</a></li>	  <li id="f-disclaimer"><a href="../../../g/e/n/Wikipedia%7EGeneral_disclaimer_3e44.html" title="Wikipedia:General disclaimer">Disclaimers</a></li>	  	</ul>
      </div>
    </div>
  </body>
</html>
