<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    		<meta name="keywords" content="Anthrax toxin,Amino acids,Anthrax,Antigen,Apoptosis,Bacillus anthracis,Bacteria,Biochemistry,CAMP,Calmodulin,Capillary" />
		<link rel="shortcut icon" href="/favicon.ico" />
		<link rel="search" type="application/opensearchdescription+xml" href="/w/opensearch_desc.php" title="Wikipedia (English)" />
		<link rel="copyright" href="../../../COPYING.html" />
    <title>Anthrax toxin - Wikipedia, the free encyclopedia</title>
    <style type="text/css">/*<![CDATA[*/ @import "../../../skins/htmldump/main.css"; /*]]>*/</style>
    <link rel="stylesheet" type="text/css" media="print" href="../../../skins/common/commonPrint.css" />
    <!--[if lt IE 5.5000]><style type="text/css">@import "../../../skins/monobook/IE50Fixes.css";</style><![endif]-->
    <!--[if IE 5.5000]><style type="text/css">@import "../../../skins/monobook/IE55Fixes.css";</style><![endif]-->
    <!--[if IE 6]><style type="text/css">@import "../../../skins/monobook/IE60Fixes.css";</style><![endif]-->
    <!--[if IE]><script type="text/javascript" src="../../../skins/common/IEFixes.js"></script>
    <meta http-equiv="imagetoolbar" content="no" /><![endif]-->
    <script type="text/javascript" src="../../../skins/common/wikibits.js"></script>
    <script type="text/javascript" src="../../../skins/htmldump/md5.js"></script>
    <script type="text/javascript" src="../../../skins/htmldump/utf8.js"></script>
    <script type="text/javascript" src="../../../skins/htmldump/lookup.js"></script>
    <script type="text/javascript" src="../../../raw/gen.js"></script>        <style type="text/css">/*<![CDATA[*/
@import "../../../raw/MediaWiki%7ECommon.css";
@import "../../../raw/MediaWiki%7EMonobook.css";
@import "../../../raw/gen.css";
/*]]>*/</style>          </head>
  <body
    class="ns-0">
    <div id="globalWrapper">
      <div id="column-content">
	<div id="content">
	  <a name="top" id="contentTop"></a>
	        <h1 class="firstHeading">Anthrax toxin</h1>
	  <div id="bodyContent">
	    <h3 id="siteSub">From Wikipedia, the free encyclopedia</h3>
	    <div id="contentSub"></div>
	    	    	    <!-- start content -->
	    <p><b>Anthrax toxin</b> refers to three <a href="../../../p/r/o/Protein.html" title="Protein">proteins</a> secreted by virulent strains of the <a href="../../../b/a/c/Bacteria.html" title="Bacteria">bacteria</a> <i><a href="../../../b/a/c/Bacillus_anthracis.html" title="Bacillus anthracis">Bacillus anthracis</a></i>. These three proteins act together in a synergistic way in which they are <a href="../../../e/n/d/Endocytosis.html" title="Endocytosis">endocytosed</a> and translocated into the <a href="../../../c/y/t/Cytoplasm.html" title="Cytoplasm">cytoplasm</a> of a <a href="../../../m/a/c/Macrophage.html" title="Macrophage">macrophage</a>, where it disrupts cellular signaling and induces <a href="../../../c/e/l/Cell_death.html" title="Cell death">cell death</a>, allowing the bacteria to evade the <a href="../../../i/m/m/Immune_system.html" title="Immune system">immune system</a>.</p>
<p>The disease known as <a href="../../../a/n/t/Anthrax.html" title="Anthrax">anthrax</a> is caused by <i><a href="../../../b/a/c/Bacillus_anthracis.html" title="Bacillus anthracis">Bacillus anthracis</a></i>, a <a href="../../../s/p/o/Spore.html" title="Spore">spore</a>-forming bacterium whose <a href="../../../p/a/t/Pathogenesis.html" title="Pathogenesis">pathogenesis</a> is primarily the result of a tripartite toxin. This toxin is composed of three proteins: the protective antigen (PA), the edema factor (EF) and the lethal factor (LF). These proteins work together to enter a cell and disrupt the signaling pathways, eventually leading to apoptosis. The molecular actions of PA, EF, and LF also provide a model biochemical system that demonstrates a variety of structure-function relationships seen in <a href="../../../b/i/o/Biochemistry.html" title="Biochemistry">biochemistry</a>.</p>
<table id="toc" class="toc" summary="Contents">
<tr>
<td>
<div id="toctitle">
<h2>Contents</h2>
</div>
<ul>
<li class="toclevel-1"><a href="#Virulence_factors_of_Bacillus_anthracis"><span class="tocnumber">1</span> <span class="toctext">Virulence factors of Bacillus anthracis</span></a></li>
<li class="toclevel-1"><a href="#Molecular_action_of_anthrax_toxin"><span class="tocnumber">2</span> <span class="toctext">Molecular action of anthrax toxin</span></a></li>
<li class="toclevel-1"><a href="#Extracellular_toxin_structure-function_relationship"><span class="tocnumber">3</span> <span class="toctext">Extracellular toxin structure-function relationship</span></a></li>
<li class="toclevel-1"><a href="#Structure-function_relationship_from_vesicle_to_cytosol"><span class="tocnumber">4</span> <span class="toctext">Structure-function relationship from vesicle to cytosol</span></a>
<ul>
<li class="toclevel-2"><a href="#Pore_formation"><span class="tocnumber">4.1</span> <span class="toctext">Pore formation</span></a></li>
<li class="toclevel-2"><a href="#Protein_translocation"><span class="tocnumber">4.2</span> <span class="toctext">Protein translocation</span></a></li>
</ul>
</li>
<li class="toclevel-1"><a href="#Questions_for_Future_Research"><span class="tocnumber">5</span> <span class="toctext">Questions for Future Research</span></a></li>
<li class="toclevel-1"><a href="#References"><span class="tocnumber">6</span> <span class="toctext">References</span></a>
<ul>
<li class="toclevel-2"><a href="#General_References"><span class="tocnumber">6.1</span> <span class="toctext">General References</span></a></li>
</ul>
</li>
</ul>
</td>
</tr>
</table>
<script type="text/javascript">
//<![CDATA[
 if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } 
//]]>
</script>
<p><a name="Virulence_factors_of_Bacillus_anthracis" id="Virulence_factors_of_Bacillus_anthracis"></a></p>
<h2><span class="editsection">[<a href="../../../a/n/t/Anthrax_toxin.html" title="Edit section: Virulence factors of Bacillus anthracis">edit</a>]</span> <span class="mw-headline">Virulence factors of <i>Bacillus anthracis</i></span></h2>
<div class="thumb tright">
<div class="thumbinner" style="width:202px;"><a href="../../../a/n/t/Image%7EAnthraxBacteria.jpg_704d.html" class="internal" title="Figure 1. Anthrax bacteria."><img src="../../../upload/shared/thumb/f/f1/AnthraxBacteria.jpg/200px-AnthraxBacteria.jpg" alt="Figure 1. Anthrax bacteria." width="200" height="157" longdesc="../../../a/n/t/Image%7EAnthraxBacteria.jpg_704d.html" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="../../../a/n/t/Image%7EAnthraxBacteria.jpg_704d.html" class="internal" title="Enlarge"><img src="../../../skins/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
Figure 1. Anthrax bacteria.</div>
</div>
</div>
<p>Anthrax is a disease caused by <i>Bacillus anthracis</i>, a spore-forming, <a href="../../../g/r/a/Gram_positive.html" title="Gram positive">Gram positive</a>, rod-shaped bacterium (Fig. 1). Its mode of host infection is through cuts, inhalation, or consumption of infected meats. The lethality of this bacterium is due to two virulent factors: its anti-<a href="../../../p/h/a/Phagocytic.html" title="Phagocytic">phagocytic</a> polysaccharide capsule (which prevents the cells of the immune system from killing the bacteria) and its secretion of the anthrax toxin (which is <a href="../../../c/y/t/Cytotoxic.html" title="Cytotoxic">cytotoxic</a> to <a href="../../../m/a/c/Macrophages.html" title="Macrophages">macrophages</a> primarily). The toxin is a mixture of three proteins: protective <a href="../../../a/n/t/Antigen.html" title="Antigen">antigen</a> (PA), <a href="../../../e/d/e/Edema.html" title="Edema">edema</a> factor (EF), and lethal factor (LF). The two virulent enzymes, EF and LF, depend on PA, which acts as a <a href="../../../t/r/o/Trojan_Horse_b8d1.html" title="Trojan Horse">Trojan Horse</a> to carry them through the <a href="../../../c/e/l/Cell_membrane.html" title="Cell membrane">plasma membrane</a> into the cell where they can make their attack.</p>
<p><a name="Molecular_action_of_anthrax_toxin" id="Molecular_action_of_anthrax_toxin"></a></p>
<h2><span class="editsection">[<a href="../../../a/n/t/Anthrax_toxin.html" title="Edit section: Molecular action of anthrax toxin">edit</a>]</span> <span class="mw-headline">Molecular action of anthrax toxin</span></h2>
<p>The three proteins of the anthrax toxin depend on each other for their toxic effect. Each protein is nontoxic on its own, but when combined, these proteins produce the lethal symptoms of anthrax. When injected in laboratory animals, the combination of EF and LF shows no effect. The combination of PA and EF, however, cause local <a href="../../../e/d/e/Edema.html" title="Edema">edema</a>, and PA mixed with LF rapidly leads to death. These studies show that these proteins work synergistically, where EF and LF depend on the presence of PA for the toxic effect.</p>
<div class="thumb tright">
<div class="thumbinner" style="width:402px;"><a href="../../../a/n/t/Image%7EAnthraxtoxins_diagram_en.png_2bdd.html" class="internal" title="Diagram of the actions of the secreted anthrax toxins."><img src="../../../upload/shared/thumb/1/14/Anthraxtoxins_diagram_en.png/400px-Anthraxtoxins_diagram_en.png" alt="Diagram of the actions of the secreted anthrax toxins." width="400" height="200" longdesc="../../../a/n/t/Image%7EAnthraxtoxins_diagram_en.png_2bdd.html" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="../../../a/n/t/Image%7EAnthraxtoxins_diagram_en.png_2bdd.html" class="internal" title="Enlarge"><img src="../../../skins/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
Diagram of the actions of the secreted anthrax toxins.</div>
</div>
</div>
<p>PA is necessary because both LF and EF function inside cells, but they are too large (90.2 kDa and 88.9 kDa, respectively) to enter via existing channels. Through a series of steps, PA helps to shuttle EF and LF into the cell (Fig. 2). This process starts when the 83 kDa PA (PA83) <a href="../../../m/o/n/Monomers.html" title="Monomers">monomers</a> bind to the largely ubiquitous human tumor <a href="../../../e/n/d/Endothelium.html" title="Endothelium">endothelium</a> marker-8 (TEM8) or <a href="../../../c/a/p/Capillary.html" title="Capillary">capillary</a> <a href="../../../m/o/r/Morphogenesis.html" title="Morphogenesis">morphogenesis</a> protein 2 (CMG2) receptors<sup id="_ref-0" class="reference"><a href="#_note-0" title="">[1]</a></sup>. Once bound, a 20 kDa N-terminal fragment (PA20) is cleaved off of PA83 by membrane endoproteases from the furin family, exposing binding sites for LF, EF, and other molecules of cleaved PA.<sup id="_ref-1" class="reference"><a href="#_note-1" title="">[2]</a></sup>. Because of this cleavage the remaining 63 kDa portion (PA63) rapidly oligomerizes to form a heptamer pre-pore,<sup id="_ref-2" class="reference"><a href="#_note-2" title="">[3]</a></sup> which then associates with up to three molecules of EF and/or LF.<sup id="_ref-3" class="reference"><a href="#_note-3" title="">[4]</a></sup> The cell then endocytoses the complex and carries it to an acidic compartment, where the low <a href="../../../p/h/_/PH_a254.html" title="PH">pH</a> causes a conformational change in the PA63 pre-pore that forms a cation-specific channel and allows the EF and LF to enter into the <a href="../../../c/y/t/Cytosol.html" title="Cytosol">cytosol</a>.<sup id="_ref-4" class="reference"><a href="#_note-4" title="">[5]</a></sup></p>
<p>Once in the cytosol, the EF and LF then carry out their respective damage-inducing processes.<sup id="_ref-5" class="reference"><a href="#_note-5" title="">[6]</a></sup> EF acts as a Ca2+ and <a href="../../../c/a/l/Calmodulin.html" title="Calmodulin">calmodulin</a> dependent adenylate <a href="../../../c/y/c/Cyclase.html" title="Cyclase">cyclase</a> that greatly increases the level of <a href="../../../c/a/m/CAMP_b3c2.html" title="CAMP">cAMP</a> in the cell. This increase in cAMP upsets water <a href="../../../h/o/m/Homeostasis.html" title="Homeostasis">homeostasis</a>, severely throws the intracellular <a href="../../../s/i/g/Signal_transduction.html" title="Signal transduction">signaling pathways</a> off balance, and impairs macrophage function, allowing the bacteria to further evade the immune system. LF also helps the bacteria evade the immune system through killing macrophages. Once in these cells, LF acts as a Zn<sup>2+</sup>-dependent <a href="../../../p/r/o/Protease.html" title="Protease">endoprotease</a> that snips off the N-terminus of <a href="../../../m/i/t/Mitogen-activated_protein_kinase.html" title="Mitogen-activated protein kinase">mitogen-activated protein kinase kinases (MAPKK)</a>. This inhibits these kinases by not allowing them to efficiently bind to their substrates, which leads to altered signaling pathways and ultimately to <a href="../../../a/p/o/Apoptosis.html" title="Apoptosis">apoptosis</a>. Thus, the synergistic effect of these three proteins leads to cellular death through a cascade of events that allow the proteins to enter the cell and disrupt cellular function.</p>
<p><a name="Extracellular_toxin_structure-function_relationship" id="Extracellular_toxin_structure-function_relationship"></a></p>
<h2><span class="editsection">[<a href="../../../a/n/t/Anthrax_toxin.html" title="Edit section: Extracellular toxin structure-function relationship">edit</a>]</span> <span class="mw-headline">Extracellular toxin structure-function relationship</span></h2>
<p>The mechanism of anthrax toxin action is the result of the molecular structures of the three toxin proteins in combination with biomolecules of the host cell. The molecular interactions are apparent upon performing a detailed analysis of the structures of PA, EF, LF, and the cellular receptors. Structures for the toxin molecules (Figs. 3–5),,, the receptor,, and for the complexes of the molecules all provided insight on the synergistic actions of these proteins. Analyses on binding sites and conformational changes augmented the structural studies, elucidating the functions of each domain of PA, LF, and EF, as briefly outlined in Table 1.</p>
<p>The structure of PA was the first to be determined (Fig. 3).<sup id="_ref-Petosa_0" class="reference"><a href="#_note-Petosa" title="">[7]</a></sup> This structure and that of its cellular receptor shed much light on the specificity of recognition and binding.<sup id="_ref-6" class="reference"><a href="#_note-6" title="">[8]</a></sup> This specificity of PA and the receptor CMG2 (similar to type I integins) is due to interactions through a metal ion dependent adhesion site (MIDAS), a hydrophobic groove, and a β-hairpin projection. These all contribute to a tight interaction in which much protein surface area on CGM2 (and TEM8) is buried.<sup id="_ref-Santelli_0" class="reference"><a href="#_note-Santelli" title="">[9]</a></sup></p>
<div class="thumb tright">
<div class="thumbinner" style="width:202px;"><a href="../../../a/n/t/Image%7EANTHRA_1.JPG_8d71.html" class="internal" title="Figure 6: Ribbon diagram of a PA63 heptamer forming a pre-pore."><img src="../../../upload/shared/thumb/9/9c/ANTHRA_1.JPG/200px-ANTHRA_1.JPG" alt="Figure 6: Ribbon diagram of a PA63 heptamer forming a pre-pore." width="200" height="194" longdesc="../../../a/n/t/Image%7EANTHRA_1.JPG_8d71.html" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="../../../a/n/t/Image%7EANTHRA_1.JPG_8d71.html" class="internal" title="Enlarge"><img src="../../../skins/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
Figure 6: Ribbon diagram of a PA<sub>63</sub> heptamer forming a pre-pore.</div>
</div>
</div>
<p>Petosa et al.solved the structure of a PA63 heptamer at 4.5 Å (0.45 nm) (Fig. 6).<sup id="_ref-Petosa_1" class="reference"><a href="#_note-Petosa" title="">[7]</a></sup> The structure they solved was of a non-membrane bound pre-pore, the conformation of the heptamer before the complex extends a β-barrel through the plasma membrane to shuttle the LF and EF into the cytosol.</p>
<p>Heptamerization and pore formation is sterically hindered by the PA20 fragment, but when it is removed from the top of the monomer, the pre-pore is quickly formed. The heptamer formation causes no major changes in the conformation of each individual monomer, but by coming together, more than 15400 Å² (154 nm²) of protein surface is buried. This buried surface consists mostly of polar or charged side groups from domains 1 and 2.<sup id="_ref-Petosa_2" class="reference"><a href="#_note-Petosa" title="">[7]</a></sup></p>
<p>During the heptamerization of PA63, molecules of EF and/or LF rapidly and simultaneously bind to the heptamer pre-pore. This binding occurs because after removing the PA20 domain, a large site is uncovered on domain 1 of PA63. Domain 1 provides a large surface that the interacts with the N-terminus of EF and LF, which is almost completely homologous for the first ~36 residues and similar in tertiary structure for the first ~250 residues.<sup id="_ref-7" class="reference"><a href="#_note-7" title="">[10]</a></sup> Studies on the binding region of LF and EF demonstrated that a large surface area contacts with domain 1 of two adjacent PA63 molecules when in the heptamer conformation.<sup id="_ref-8" class="reference"><a href="#_note-8" title="">[11]</a></sup> This large binding area explains why previous studies could only bind up to three molecules on a PA63 heptamer. The LF/EF binding site is now being utilized for delivery of therapeutics via fusion proteins.</p>
<p>Upon formation of the prepore and attachment of LF and/or EF, the heptamer migrates to a lipid raft where it is rapidly endocytosed. <a href="../../../e/n/d/Endocytosis.html" title="Endocytosis">Endocytosis</a> occurs as a result of a series of events. This begins when CGM2 or TEM8 is palmitoylated, which inhibits the association of the receptor with lipid rafts. This inhibits the receptor from being endocytosed before PA83 is cleaved and before LF or EF can associate with the heptamer. Reassociation of the receptor with the <a href="../../../c/h/o/Cholesterol.html" title="Cholesterol">cholesterol</a> and glycosphigolipid-rich microdomains (<a href="../../../l/i/p/Lipid_rafts.html" title="Lipid rafts">lipid rafts</a>) occurs when PA63 binds to the receptor and heptamerizes. Once the receptor and PA returns to the lipid raft, E3 ubiquitin ligase Cb1 ubiquitinates the cytoplasmic tail of the receptor, signaling the receptor and associated toxin proteins for endocytosis. <a href="../../../d/y/n/Dynamin.html" title="Dynamin">Dynamin</a> and Eps15 are required for this endocytosis to occur, indicating that anthrax toxin enters the cell via the <a href="../../../c/l/a/Clathrin.html" title="Clathrin">clathrin</a>-dependent pathway.<sup id="_ref-9" class="reference"><a href="#_note-9" title="">[12]</a></sup></p>
<p>As discussed, each molecule interacts with several others in order to induce the endocytosis of the anthrax toxin. Once inside, the complex is transferred to an acidic compartment, where the heptamer, still in the non-membrane-spanning pre-pore conformation, is prepared for translocation of EF and LF into the cytosol.<sup id="_ref-10" class="reference"><a href="#_note-10" title="">[13]</a></sup></p>
<p><a name="Structure-function_relationship_from_vesicle_to_cytosol" id="Structure-function_relationship_from_vesicle_to_cytosol"></a></p>
<h2><span class="editsection">[<a href="../../../a/n/t/Anthrax_toxin.html" title="Edit section: Structure-function relationship from vesicle to cytosol">edit</a>]</span> <span class="mw-headline">Structure-function relationship from vesicle to cytosol</span></h2>
<p><a name="Pore_formation" id="Pore_formation"></a></p>
<h3><span class="editsection">[<a href="../../../a/n/t/Anthrax_toxin.html" title="Edit section: Pore formation">edit</a>]</span> <span class="mw-headline">Pore formation</span></h3>
<p>At first glance, the primary sequence of PA does not look like a membrane-spanning protein. A <a href="../../../h/y/d/Hydrophobicity.html" title="Hydrophobicity">hydrophobicity</a> plot lacked any patterns which are common of possible membrane-spanning domains. The structures other multimeric membrane proteins (such as dyptheria toxin) provide the answer to how the membrane spanning reason it. It is thought that PA acted like these multimeric membrane proteins, containing a β-barrel, made with a stretch of <a href="../../../a/m/i/Amino_acids.html" title="Amino acids">amino acids</a> (both polar and non-polar) from each monomer.<sup id="_ref-Petosa_3" class="reference"><a href="#_note-Petosa" title="">[7]</a></sup></p>
<div class="thumb tright">
<div class="thumbinner" style="width:102px;"><a href="../../../a/n/t/Image%7EAnthrax_toxin_protein_key_motif.jpg_449d.html" class="internal" title="Greek-key motif."><img src="../../../upload/shared/thumb/d/d8/Anthrax_toxin_protein_key_motif.jpg/100px-Anthrax_toxin_protein_key_motif.jpg" alt="Greek-key motif." width="100" height="96" longdesc="../../../a/n/t/Image%7EAnthrax_toxin_protein_key_motif.jpg_449d.html" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="../../../a/n/t/Image%7EAnthrax_toxin_protein_key_motif.jpg_449d.html" class="internal" title="Enlarge"><img src="../../../skins/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
Greek-key motif.</div>
</div>
</div>
<p>The formation of the β-barrel pore is facilitated with a drop in pH. To form the barrel when the pH drops, PA63 domain 2 must undergo the greatest conformation change. Upon examination of the structure of domain 2 (Fig. 7), one can see that this domain contains a Greek-key motif (the gold portion in Fig. 7). A general schematic of a Greek-key motif is shown in Fig. 8. Attached to the Greek key in domain 2 is a large disordered loop. The necessity of this loop in pore formation is shown through using mutagenesis and proteolysis of the loop with chymotrypsin. Additional electrophysiological measurements of cysteine substitutions place the amino acids of this loop inside the lumen of the membrane inserted pore. The disordered loop in domain 2 also has a pattern of alternating hydrophobic and hydrophilic amino acids, which is a pattern conserved in the membrane-spanning portions of porins. The only problem is that the loop is not large enough to span a membrane in a β-barrel. This membrane insertion could only occur with additional conformational changes. A large conformational change does takes place where the Greek-key motif unfolds, forming a β-hairpin that projects downward into the membrane and forms a β-barrel with the other 6 monomers of the complex (figures 9a and 9b). The final pore has a diameter of 12 Å (1.2 nm), which fits the theoretical value of this model.<sup id="_ref-Petosa_4" class="reference"><a href="#_note-Petosa" title="">[7]</a></sup></p>
<p>This model would require large conformational changes in domain 2 along with the breaking of many hydrogen bonds as the Greek-key motif peels away from the center of the domain. Petosa et al. proposed a model of how this occurs.<sup id="_ref-Petosa_5" class="reference"><a href="#_note-Petosa" title="">[7]</a></sup> Insertion of the PA Greek key motifs into the membrane occurs when the heptamer is acidified. On artificial bilayers, this occurs when the pH is dropped from 7.4 to 6.5, suggesting that the trigger for insertion involves a titration of histidines. This indeed fits the sequence of PA since domain 2 contains a number of histidines (shown as asterisks in figure 9a). Three histidines are found in the disordered loop, one of which lies with a Greek-key histidine within a cluster of polar amino acids. This cluster (including the two histidines, three arginines and one glutamate) is imbedded at the top of the Greek-key motif, so it is easy to see that the protonation of these histidines would disrupt the cluster. Furthermore, another histidine is located at the base of the Greek-key motif along with a number of hydrophobic residues (on the green segment in figures 7 and 9a). At pH 7.4 this segment is ordered, but when the crystals are grown at pH 6.0, it becomes disordered. This order to disorder transition is the initial step of PA membrane insertion.</p>
<p>PA is endocytosed as a soluble heptamer attached to its receptors, with LF or EF attached to the heptamer as cargo. The first step after endocytosis is the acidification of the endocytotic vesicle. The acidification plays two roles in the lifespan of the toxin. First, it helps to relax the tight grip of the CMG2 or TEM8 receptor on PA, facilitating the pore formation (the different receptors allow for insertion at a slightly different pH).<sup id="_ref-Santelli_1" class="reference"><a href="#_note-Santelli" title="">[9]</a></sup> Second, the drop in pH causes a disordered loop and a Greek-key motif in the PA domain 2 to fold out of the heptamer pre-pore and insert through the wall of the acidic vesicle, leading to pore formation (Figures 7–9).</p>
<p>Santelli et al. explained more about the process after they determined the crystal structure of the PA/CMG2 complex.<sup id="_ref-Santelli_2" class="reference"><a href="#_note-Santelli" title="">[9]</a></sup> The structure of this complex shows the binding of CMG2 by both domain 2 and 4 of PA. This interaction demonstrates less freedom to unfold the Greek key. Further analysis shows that seven of the nine histidines in PA are on the domain 2/domain 4 interface. Protonation of these histidines causes the domains to separate enough to allow the Greek-key to flop out and help form the β-hairpin involved in insertion. In addition, when PA binds to CMG2, insertion no longer occurs at a pH of 6.5, as it does when inserted into an artificial membrane. Instead it requires a pH of 5.0 for insertion in natural cells. This difference was explained to be the result of the pocket next to the MIDAS motif in CMG2. This pocket contains a histidine buried at the bottom where domain 2 attaches. This histidine is protonated at a lower pH and adds greater stability to PA. This added stability keeps the Greek-key from being able to move until more acidic conditions are met. These histidines all work in conjunction to keep the heptamer from inserting prematurely before endocytosis occurs.</p>
<p>Santelli and colleagues (Fig. 10) also built a hypothetical structure of the membrane-inserted PA/CMG2 structure. This model shows that the β-barrel is about 70 Å (7 nm) long, 30 Å (3 nm) of which span the membrane and the 40 Å (4 nm) gap is actually filled in with the rest of the extracellular portion of the CMG2 receptor (~100 residues). CMG2 provides additional support to the pore.</p>
<p><a name="Protein_translocation" id="Protein_translocation"></a></p>
<h3><span class="editsection">[<a href="../../../a/n/t/Anthrax_toxin.html" title="Edit section: Protein translocation">edit</a>]</span> <span class="mw-headline">Protein translocation</span></h3>
<div class="thumb tright">
<div class="thumbinner" style="width:302px;"><a href="../../../t/r/a/Image%7ETranslocation_of_anthrax_toxin_protein.jpg_c839.html" class="internal" title="Diagram of protein translocation."><img src="../../../upload/shared/thumb/a/ab/Translocation_of_anthrax_toxin_protein.jpg/300px-Translocation_of_anthrax_toxin_protein.jpg" alt="Diagram of protein translocation." width="300" height="161" longdesc="../../../t/r/a/Image%7ETranslocation_of_anthrax_toxin_protein.jpg_c839.html" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="../../../t/r/a/Image%7ETranslocation_of_anthrax_toxin_protein.jpg_c839.html" class="internal" title="Enlarge"><img src="../../../skins/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
Diagram of protein translocation.</div>
</div>
</div>
<p>Several recent studies demonstrate how the PA63 pore allows the EF and LF into the cytoplasm when its lumen is so small. The lumen on the PA63 pore is only 15 Å (1.5 nm) across, which is much smaller than the diameter of LF or EF. Translocation occurs through a series of events which begin in the endosome as it acidifies. LF and EF are pH sensitive, and as the pH drops, their structures lose stability. Below a pH of 6.0 (the pH in an endosome), both LF and EF become disordered molten globules. When a molecule is in this conformation, the N-terminus is freed and drawn into the pore by the proton gradient and positive transmembrane potential. A ring of seven phenylalanines at the mouth endosome side of the pore (phenylalanine clamp) assists in the unfolding of LF or EF by interacting with the hydrophobic residues found in LF or EF. The proton gradient then begins to lace the protein though the pore. The lacing mechanism is driven by the gradient, but requires the phenylalanine clamp for a ratcheting motion. The first 250 residues of EF and LF have an irregular alternating sequence of basic, acidic, and hydrophobic residues. The interplay between the phenylalanine clamp and the protonation state cause a ratcheting effect that drives the protein though until enough has crossed into the cytoplasm to drag the rest through the pore as the N-terminus refolds (Fig. 11).</p>
<p><a name="Questions_for_Future_Research" id="Questions_for_Future_Research"></a></p>
<h2><span class="editsection">[<a href="../../../a/n/t/Anthrax_toxin.html" title="Edit section: Questions for Future Research">edit</a>]</span> <span class="mw-headline">Questions for Future Research</span></h2>
<p>Despite the recent advances in the understanding of anthrax toxin, there are still several missing details in the action of anthrax toxin. These missing details leave questions about the molecular actions inside the cell.2 What role does EF play in hindering the immune system? Does it work with LF for its effect? How do the enzymes refold after translocation? Is there a <a href="../../../c/h/a/Chaperonin.html" title="Chaperonin">chaperonin</a>? Two proteins: Kif1C and the <a href="../../../p/r/o/Proteasome.html" title="Proteasome">proteasome</a> have shown a contribution to the effect of lethal toxin, but how do they contribute? Does LF target certain MAPKKs with a greater specificity? Does LF target other molecules too?</p>
<p><a name="References" id="References"></a></p>
<h2><span class="editsection">[<a href="../../../a/n/t/Anthrax_toxin.html" title="Edit section: References">edit</a>]</span> <span class="mw-headline">References</span></h2>
<div class="references-small" style="-moz-column-count:2; column-count:2;">
<ol class="references">
<li id="_note-0"><b><a href="#_ref-0" title="">^</a></b> <cite style="font-style:normal">Sternbach, G. (2003). "The history of anthrax". <i>J Emerg Med</i> <b>24</b>: 463-467.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.atitle=The+history+of+anthrax&amp;rft.title=J+Emerg+Med&amp;rft.jtitle=J+Emerg+Med&amp;rft.date=2003&amp;rft.volume=24&amp;rft.au=Sternbach%2C+G.&amp;rft.pages=463-467">&#160;</span></li>
<li id="_note-1"><b><a href="#_ref-1" title="">^</a></b> <cite style="font-style:normal">Abrami L, Reig N, van der Goot FG (2005). "Anthrax toxin: the long and winding road that leads to the kill". <i>Trends Microbiol</i> <b>13</b>: 72-78.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.atitle=Anthrax+toxin%3A+the+long+and+winding+road+that+leads+to+the+kill&amp;rft.title=Trends+Microbiol&amp;rft.jtitle=Trends+Microbiol&amp;rft.date=2005&amp;rft.volume=13&amp;rft.au=Abrami+L%2C+Reig+N%2C+van+der+Goot+FG&amp;rft.pages=72-78">&#160;</span></li>
<li id="_note-2"><b><a href="#_ref-2" title="">^</a></b> <cite style="font-style:normal">Green BD, Battisti L, Koehler TM, Throne CB, Ivins BE (1985). "Demonstration of a capsule plasmid in Bacillus anthracis". <i>Infect Immun</i> <b>49</b>: 291-297.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.atitle=Demonstration+of+a+capsule+plasmid+in+Bacillus+anthracis&amp;rft.title=Infect+Immun&amp;rft.jtitle=Infect+Immun&amp;rft.date=1985&amp;rft.volume=49&amp;rft.au=Green+BD%2C+Battisti+L%2C+Koehler+TM%2C+Throne+CB%2C+Ivins+BE&amp;rft.pages=291-297">&#160;</span></li>
<li id="_note-3"><b><a href="#_ref-3" title="">^</a></b> <cite style="font-style:normal">Grinberg LM, Abramova FA, Yampolskaya OV, Walker DH, Smith JH (2001). "Quantitative pathology of inhalational anthrax I: quantitative microscopic findings". <i>Mod Pathol</i> <b>14</b>: 482-495.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.atitle=Quantitative+pathology+of+inhalational+anthrax+I%3A+quantitative+microscopic+findings&amp;rft.title=Mod+Pathol&amp;rft.jtitle=Mod+Pathol&amp;rft.date=2001&amp;rft.volume=14&amp;rft.au=Grinberg+LM%2C+Abramova+FA%2C+Yampolskaya+OV%2C+Walker+DH%2C+Smith+JH&amp;rft.pages=482-495">&#160;</span></li>
<li id="_note-4"><b><a href="#_ref-4" title="">^</a></b> <cite style="font-style:normal">Friedlander AM, Bhatnagar R, Leppla SH, Johnson L, Singh Y (1993). "Characterization of macrophage sensitivity and resistance to anthrax lethal toxin". <i>Infect Immun</i> <b>61</b>: 245-252.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.atitle=Characterization+of+macrophage+sensitivity+and+resistance+to+anthrax+lethal+toxin&amp;rft.title=Infect+Immun&amp;rft.jtitle=Infect+Immun&amp;rft.date=1993&amp;rft.volume=61&amp;rft.au=Friedlander+AM%2C+Bhatnagar+R%2C+Leppla+SH%2C+Johnson+L%2C+Singh+Y&amp;rft.pages=245-252">&#160;</span></li>
<li id="_note-5"><b><a href="#_ref-5" title="">^</a></b> <cite style="font-style:normal">Singh Y, Leppla SH, Bhatnagar R, Friedlander AM. (1989). "Internalization and processing of Bacillus anthracis lethal toxin by toxin-sensitive and -resistant cells". <i>J Biol Chem</i> <b>264</b>: 11099-11102.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.atitle=Internalization+and+processing+of+Bacillus+anthracis+lethal+toxin+by+toxin-sensitive+and+-resistant+cells&amp;rft.title=J+Biol+Chem&amp;rft.jtitle=J+Biol+Chem&amp;rft.date=1989&amp;rft.volume=264&amp;rft.au=Singh+Y%2C+Leppla+SH%2C+Bhatnagar+R%2C+Friedlander+AM.&amp;rft.pages=11099-11102">&#160;</span></li>
<li id="_note-Petosa">^ <a href="#_ref-Petosa_0" title=""><sup><i><b>a</b></i></sup></a> <a href="#_ref-Petosa_1" title=""><sup><i><b>b</b></i></sup></a> <a href="#_ref-Petosa_2" title=""><sup><i><b>c</b></i></sup></a> <a href="#_ref-Petosa_3" title=""><sup><i><b>d</b></i></sup></a> <a href="#_ref-Petosa_4" title=""><sup><i><b>e</b></i></sup></a> <a href="#_ref-Petosa_5" title=""><sup><i><b>f</b></i></sup></a> Petosa, C.; Collier, R. J.; Klimpel, K. R.; Leppla, S. H.; Liddington, R. C. Crystal structure of the anthrax toxin protective antigen. Nature. 1997, 385, 833–838.</li>
<li id="_note-6"><b><a href="#_ref-6" title="">^</a></b> Lacy, D. B.; Wigelsworth, D. J.; Scobie, H. M.; Young, J. A.; Collier, R. J. Crystal structure of the von Willebrand factor A domain of human capillary morphogenesis protein 2: an anthrax toxin receptor. Proc. Natl. Acad. Sci. U.S.A. 2004, 101, 6367–6372.</li>
<li id="_note-Santelli">^ <a href="#_ref-Santelli_0" title=""><sup><i><b>a</b></i></sup></a> <a href="#_ref-Santelli_1" title=""><sup><i><b>b</b></i></sup></a> <a href="#_ref-Santelli_2" title=""><sup><i><b>c</b></i></sup></a> Santelli, E.; Bankston, L. A.; Leppla, S. H.; Liddington, R. C. Crystal structure of a complex between anthrax toxin and its host cell receptor. Nature. 2004, 430, 905–908.</li>
<li id="_note-7"><b><a href="#_ref-7" title="">^</a></b> Pannifer, A. D.; Wong, T. Y.; Schwarzenbacher, R.; Renatus, M.; Petosa, C.; Bienkowska, J.; Lacy, D. B.; Collier, R. J.; Park, S.; Leppla, S. H.; Hanna, P.; Liddington, R. C. Crystal structure of the anthrax lethal factor. Nature. 2001, 414, 230–233.</li>
<li id="_note-8"><b><a href="#_ref-8" title="">^</a></b> Melnyk, R. A.; Hewitt, K. M.; Lacy, D. B.; Lin, H. C.; Gessner, C. R.; Li, S.; Woods, V. L.; Collier, R. J. Structural Determinates for the Binding of Anthrax Lethal Factor to Oligomeric Protective Antigen. J. Biol. Chem. 2006, 281, 1630–1635.</li>
<li id="_note-9"><b><a href="#_ref-9" title="">^</a></b> Abrami, L.; Liu, S.; Cosson, P.; Leppla, S. H.; van der Goot, F. G. Anthrax toxin triggers endocytosis of its receptor via a lipid raft-mediated clathrin-dependent process. J. Cell Biol. 2003, 160, 321–328.</li>
<li id="_note-10"><b><a href="#_ref-10" title="">^</a></b> Mourez, M. Anthrax toxins. Rev. Physiol. Biochem. Pharmacol. 2004, 152, 135–164.</li>
</ol>
</div>
<p><a name="General_References" id="General_References"></a></p>
<h3><span class="editsection">[<a href="../../../a/n/t/Anthrax_toxin.html" title="Edit section: General References">edit</a>]</span> <span class="mw-headline">General References</span></h3>
<div class="references-small" style="-moz-column-count:2; column-count:2;">
<ul>
<li>Smith, H.; Stoner, H. B. Anthrax toxic complex. Fed. Proc. 1967, 26, 1554–1557.</li>
<li>Öncü, S.; Öncü, S.; Sakarya1, S. Anthrax – an overview. Med. Sci. Monit. 2003, 9, 276–283.</li>
<li>Stanley, J. L.; Smith, H. Purification of factor I and recognition of a third factor of anthrax toxin. J. Gen. Microbiol. 1961, 26, 49–66.</li>
<li>Beall, F. A.; Taylor, M. J.; Thorne, C. B. Rapid lethal effects in rats of a third component found upon fractionating the toxin of Bacillus anthracis. J. Bacteriol. 1962, 83, 1274–1280.</li>
<li>Baron, S. Medical Microbiology, 4th ed. Addison-Wesley Publishing Company, Inc. 1996.</li>
<li>Tonello, F.; Ascenzi, P.; Montecucco, C. The metalloproteolytic activity of the anthrax lethal factor is substrate-inhibited. J. Biol. Chem. 2003, 278, 40075–40078.</li>
<li>Bradley, K. A.; Mogridge, J.; Mourez, M.; Collier, R. J.; Young, J. A. T. Identification of the cellular receptor for anthrax toxin. Nature. 2001, 414, 225–229.</li>
<li>Scobie, H. M.; Rainey, G. J. A.; Bradley, K. A.; Young, J. A. T. Human capillary morphogenesis protein 2 functions as an anthrax toxin receptor. Proc. Natl. Acad. Sci. USA. 2003, 100, 5170–5174.</li>
<li>Milne, J. C.; Furlong, D.; Hanna, P. C.; Wall, J. S.; Collier, R. J. Anthrax protective antigen forms oligomers during intoxication of mammalian cells. J. Biol. Chem. 1994, 269, 20607–20612.</li>
<li>Mogridge, J.; Cunningham, K.; Collier, R. J. Stoichiometry of anthrax toxin complexes. Biochemistry. 2002, 41, 1079–1082.</li>
<li>Leppla, S. H. Anthrax toxin edema factor: a bacterial adenylate cyclase that increases cyclic AMP concentrations of eukaryotic cells. Proc. Natl. Acad. Sci. U.S.A. 1982, 79, 3162–3166.</li>
<li>O'Brien, J.; Friedlander, A.; Dreier, T.; Ezzell, J.; Leppla, S. Effects of anthrax toxin components on human neutrophils. Infect. Immun. 1985, 47, 306–310.</li>
<li>Orlik, F.; Schiffler, B.; Benz, R. Anthrax toxin protective antigen: inhibition of channel function by chloroquine and related compounds and study of binding kinetics using the current noise analysis. Biophys. J. 2005, 88, 1715–1724.</li>
<li>Drum, C. L.; Shen, Y.; Rice, P. A.; Bohm, A.; Tang, W. J. Crystallization and preliminary X-ray study of the edema factor exotoxin adenylyl cyclase domain from Bacillus anthracis in the presence of its activator, calmodulin. Acta Crystallogr. D Biol. Crystallogr. 2005, 57, 1881–1884.</li>
<li>Drum, C. L.; Yan, S. Z.; Bard, J.; Shen, Y. Q.; Lu, D.; Soelaiman, S.; Grabarek, Z.; Bohm A.; Tang, W. J. Structural basis for the activation of anthrax adenylyl cyclase exotoxin by calmodulin. Nature. 2002, 415, 396–402.</li>
<li>Lacy, D. B.; Lin, H. C.; Melnyk, R. A.; Schueler-Furman, O.; Reither, L.; Cunningham, K.; Baker, D.; Collier, R. J. A model of anthrax toxin lethal factor bound to protective antigen. Proc. Natl. Acad. Sci. U.S.A. 2005, 102, 16409–16414.</li>
<li>Lacy, D. B.; Mourez, M.; Fouassier, A.; Collier, R. J. Mapping the anthrax protective antigen binding site on the lethal and edema factors. J. Biol. Chem. 2002, 277, 3006–3010.</li>
<li>Arora, N; Leppla, S. H. Residues 1-254 of anthrax toxin lethal factor are sufficient to cause cellular uptake of fused polypeptides. J. Biol. Chem. 1993, 268, 3334–3341.</li>
<li>Leppla, S. H.; Arora, N; Varughese, M. Anthrax toxin fusion proteins for intracellular delivery of macromolecules. J. Appl. Microbiol. 1999, 87, 284.</li>
<li>Goletz, T. J.; Klimpel, K. R.; Arora, N.; Leppla, S. H.; Keith, J. M.; Berzofsky, J. A.; Targeting HIV proteins to the major histocompatibility complex class I processing pathway with a novel gp120-anthrax toxin fusion protein. Proc. Natl. Acad. Sci. USA. 1997, 94, 12059–12064.</li>
<li>Collier, R. J.; Young, J.A.T. Anthrax Toxin. Annu. Rev. Cell Dev. Biol. 2003, 19, 45–70.</li>
<li>Abrami, L.; Leppla, S. H.; van der Goot, F. G. Receptor palmitoylation and ubiquitination regulate anthrax toxin endocytosis. J. Cell Biol. 2006, 172, 309–320.</li>
<li>Singh, Y.; Klimpel, K. R.; Arora, N.; Sharma, M.; Leppla, S. H. The chymotrypsin- sensitive site, FFD315, in anthrax toxin protective antigen is required for translocation of lethal factor. J. Biol. Chem. 1994, 269, 29039–29046.</li>
<li>Benson, E. L.; Huynh, P. D.; Finkelstein, A.; Collier, R. J. Identification of residues lining the anthrax protective antigen channel. Biochemistry. 1998, 37, 3941–3948.</li>
<li>Blaustein, R.O.; Koehler, T.M.; Collier, R.J.; Finkelstein, A. Anthrax toxin: channel-forming activity of protective antigen in planar phospholipid bilayers. Proc. Natl. Acad. Sci. USA. 1989, 86, 2209–22013.</li>
<li>Wolfe, J. T.; Krantz, B. A.; Rainey, G. J. A.; Young, J. A. T.; Collier, R. J. Whole-cell voltage clamp measurements of anthrax toxin pore current. J. Biol. Chem. 2005, 280, 39417–39422.</li>
<li>Milne, J. C.; Collier, R.J. pH-dependent permeabilization of the plasma membrane of mammalian cells by anthrax protective antigen. Mol. Microbiol. 1993, 10, 647–653.</li>
<li>Krantz, B. A.; Trivedi, A. D.; Cunningham, K.; Christensen K. A.; Collier, R. J. Acid-induced Unfolding of the Amino-terminal Domains of the Lethal and Edema Factors of Anthrax Toxin. J. Mol. Biol. 2004, 344, 739–756.</li>
<li>Zhang, S.; Finkelstein, A.; Collier, R. J. Evidence that translocation of anthrax toxin's lethal factor is initiated by entry of its N terminus into the protective antigen channel. Proc. Natl. Acad. Sci. U.S.A. 2004, 101, 16756–16761</li>
<li>Zhang, S.; Udho, E.; Wu, Z.; Collier, R. J.; Finkelstein, A. Protein translocation through anthrax toxin channels formed in planar lipid bilayers. Biophys J. 2004, 87, 3842–3849.</li>
<li>Krantz, B. A.; Melnyk, R. A.; Zhang, S.; Juris, S. J.; Lacy, D. B.; Wu, Z.; Finkelstein, A.; Collier, R. J. A phenylalanine clamp catalyzes protein translocation through the anthrax toxin pore. Science. 2005, 309, 777–781.</li>
<li>Krantz, B. A.; Finkelstein, A.; Collier, R. J. Protein translocation through the anthrax toxin transmembrane pore is driven by a proton gradient. J. Mol. Biol. 2006, 355, 968–979.</li>
<li>Watters, J. W.; Dewar, K.; Lehoczky, J.; Boyartchuk, V.; Dietrich, W.F. Kif1C, a kinesin-like motor protein, mediates mouse macrophage resistance to anthrax lethal factor. Curr. Biol. 2001, 11, 1503–1511.</li>
<li>Tang, G.; Leppla, S. H. Proteasome activity is required for anthrax lethal toxin to kill macrophages. Infect. Immun. 1999, 67, 3055–3060.</li>
</ul>
</div>

<!-- 
Pre-expand include size: 27395 bytes
Post-expand include size: 9479 bytes
Template argument size: 7128 bytes
Maximum: 2048000 bytes
-->
<div class="printfooter">
Retrieved from "<a href="http://en.wikipedia.org../../../a/n/t/Anthrax_toxin.html">http://en.wikipedia.org../../../a/n/t/Anthrax_toxin.html</a>"</div>
	    <div id="catlinks"><p class='catlinks'><a href="../../../c/a/t/Special%7ECategories_101d.html" title="Special:Categories">Categories</a>: <span dir='ltr'><a href="../../../b/a/c/Category%7EBacterial_diseases_0121.html" title="Category:Bacterial diseases">Bacterial diseases</a></span> | <span dir='ltr'><a href="../../../p/r/o/Category%7EProteins_5bd5.html" title="Category:Proteins">Proteins</a></span> | <span dir='ltr'><a href="../../../c/h/e/Category%7EChemical_weapons_fb0c.html" title="Category:Chemical weapons">Chemical weapons</a></span></p></div>	    <!-- end content -->
	    <div class="visualClear"></div>
	  </div>
	</div>
      </div>
      <div id="column-one">
	<div id="p-cactions" class="portlet">
	  <h5>Views</h5>
	  <ul>
	    <li id="ca-nstab-main"
	       class="selected"	       ><a href="../../../a/n/t/Anthrax_toxin.html">Article</a></li><li id="ca-talk"
	       	       ><a href="../../../a/n/t/Talk%7EAnthrax_toxin_042d.html">Discussion</a></li><li id="ca-current"
	       	       ><a href="http://en.wikipedia.org/wiki/Anthrax_toxin">Current revision</a></li>	  </ul>
	</div>
	<div class="portlet" id="p-logo">
	  <a style="background-image: url(../../../images/wiki-en.png);"
	    href="../../../index.html"
	    title="Main Page"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
		<div class='portlet' id='p-navigation'>
	  <h5>Navigation</h5>
	  <div class='pBody'>
	    <ul>
	    	      <li id="n-Main-page"><a href="../../../index.html">Main page</a></li>
	     	      <li id="n-Contents"><a href="../../../c/o/n/Wikipedia%7EContents_3181.html">Contents</a></li>
	     	      <li id="n-Featured-content"><a href="../../../f/e/a/Wikipedia%7EFeatured_content_24ba.html">Featured content</a></li>
	     	      <li id="n-currentevents"><a href="../../../c/u/r/Portal%7ECurrent_events_bb60.html">Current events</a></li>
	     	    </ul>
	  </div>
	</div>
		<div class='portlet' id='p-interaction'>
	  <h5>interaction</h5>
	  <div class='pBody'>
	    <ul>
	    	      <li id="n-About-Wikipedia"><a href="../../../a/b/o/Wikipedia%7EAbout_8d82.html">About Wikipedia</a></li>
	     	      <li id="n-portal"><a href="../../../c/o/m/Wikipedia%7ECommunity_Portal_6a3c.html">Community portal</a></li>
	     	      <li id="n-contact"><a href="../../../c/o/n/Wikipedia%7EContact_us_afd6.html">Contact us</a></li>
	     	      <li id="n-sitesupport"><a href="http://wikimediafoundation.org/wiki/Fundraising">Make a donation</a></li>
	     	      <li id="n-help"><a href="../../../c/o/n/Help%7EContents_22de.html">Help</a></li>
	     	    </ul>
	  </div>
	</div>
		<div id="p-search" class="portlet">
	  <h5><label for="searchInput">Search</label></h5>
	  <div id="searchBody" class="pBody">
	    <form action="javascript:goToStatic(3)" id="searchform"><div>
	      <input id="searchInput" name="search" type="text"
	        accesskey="f" value="" />
	      <input type='submit' name="go" class="searchButton" id="searchGoButton"
	        value="Go" />
	    </div></form>
	  </div>
	</div>
	      </div><!-- end of the left (by default at least) column -->
      <div class="visualClear"></div>
      <div id="footer">
    <div id="f-poweredbyico"><a href="http://www.mediawiki.org/"><img src="../../../skins/common/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" /></a></div>	<div id="f-copyrightico"><a href="http://wikimediafoundation.org/"><img src="../../../images/wikimedia-button.png" border="0" alt="Wikimedia Foundation"/></a></div>	<ul id="f-list">
	  	  	  <li id="f-credits">This page was last modified 03:43, 26 March 2007 by Wikipedia user <a href="../../../j/a/y/User%7EJayron32_634d.html" title="User:Jayron32">Jayron32</a>. Based on work by Wikipedia user(s) Smeeto, <a href="../../../n/a/p/User%7ENapalmSunday_a9e1.html" title="User:NapalmSunday">NapalmSunday</a>, <a href="../../../d/o/1/User%7EDO11.10_882e.html" title="User:DO11.10">DO11.10</a>, <a href="../../../d/a/v/User%7EDavidruben_76a6.html" title="User:Davidruben">Davidruben</a>, <a href="../../../b/o/b/User%7EBobo192_c260.html" title="User:Bobo192">Bobo192</a>, <a href="../../../w/h/p/User%7EWhpq_e938.html" title="User:Whpq">Whpq</a>, <a href="../../../s/e/d/User%7ESedmic_6a9b.html" title="User:Sedmic">Sedmic</a>, <a href="../../../n/a/t/User%7ENatelewis_4f3d.html" title="User:Natelewis">Natelewis</a>, <a href="../../../s/t/e/User%7EStevenfruitsmaak_e31a.html" title="User:Stevenfruitsmaak">Stevenfruitsmaak</a>, <a href="../../../g/e/n/User%7EGene_Nygaard_634f.html" title="User:Gene Nygaard">Gene Nygaard</a>, <a href="../../../p/e/a/User%7EPearle_6f07.html" title="User:Pearle">Pearle</a>, <a href="../../../d/a/v/User%7EDavidfstr_86a5.html" title="User:Davidfstr">Davidfstr</a>, <a href="../../../o/h/c/User%7EOhconfucius_bd0f.html" title="User:Ohconfucius">Ohconfucius</a>, <a href="../../../p/i/e/User%7EPietdesomere_7597.html" title="User:Pietdesomere">Pietdesomere</a>, <a href="../../../a/s/p/User%7EAsparagus_6351.html" title="User:Asparagus">Asparagus</a> and <a href="../../../c/m/d/User%7ECmdrObot_0605.html" title="User:CmdrObot">CmdrObot</a> and Anonymous user(s) of Wikipedia.</li>	  <li id="f-copyright">All text is available under the terms of the <a class='internal' href="../../../t/e/x/Wikipedia%7EText_of_the_GNU_Free_Documentation_License_702a.html" title="Wikipedia:Text of the GNU Free Documentation License">GNU Free Documentation License</a>. (See <b><a class='internal' href="../../../c/o/p/Wikipedia%7ECopyrights_92c4.html" title="Wikipedia:Copyrights">Copyrights</a></b> for details.) <br /> Wikipedia&reg; is a registered trademark of the <a href="http://www.wikimediafoundation.org">Wikimedia Foundation, Inc</a>., a US-registered <a class='internal' href="../../../5/0/1/501%28c%29.html#501.28c.29.283.29" title="501(c)(3)">501(c)(3)</a> <a href="http://wikimediafoundation.org/wiki/Deductibility_of_donations">tax-deductible</a> <a class='internal' href="../../../n/o/n/Non-profit_organization.html" title="Non-profit organization">nonprofit</a> <a href="../../../c/h/a/Charitable_organization.html" title="Charitable organization">charity</a>.<br /></li>	  <li id="f-about"><a href="../../../a/b/o/Wikipedia%7EAbout_8d82.html" title="Wikipedia:About">About Wikipedia</a></li>	  <li id="f-disclaimer"><a href="../../../g/e/n/Wikipedia%7EGeneral_disclaimer_3e44.html" title="Wikipedia:General disclaimer">Disclaimers</a></li>	  	</ul>
      </div>
    </div>
  </body>
</html>
